[Publication list on the CeBiTec website](https://www.cebitec.uni-bielefeld.de/~bpucker#publications)

[Google Scholar Profile](https://scholar.google.com/citations?user=GnnyChAAAAAJ&hl=en)



# **Publications Boas Pucker**

### 2021

Frey, K., Weisshaar, B. and **Pucker, B.** (2021). Reference-based QUantification Of gene Dispensability (QUOD). Plant Methods 17, 18. doi:[10.1186/s13007-021-00718-5](https://doi.org/10.1186/s13007-021-00718-5).


Xu B., Taylor L., **Pucker B.**, Feng T., Glover B. J. and Brockington S. (2021). The land plant-specific MIXTA-MYB lineage is implicated in the early evolution of the plant cuticle and the colonization of land. New Phytologist. doi:[10.1111/nph.16997](https://doi.org/10.1111/nph.16997).



## 2020

**Pucker B.**, Schwandner A., Becker S., Hausmann L., Viehöver P., Töpfer R., Weisshaar B., Holtgräwe D. (2020). RNA-Seq Time Series of Vitis vinifera Bud Development Reveals Correlation of Expression Patterns with the Local Temperature Profile. Plants 2020, 9(11), 1548; doi:[10.3390/plants9111548](https://www.mdpi.com/2223-7747/9/11/1548/htm).

**Pucker B.** and Brockington S. (2020). Pitaya transcriptome assemblies and investigation of transcript abundances. Bielefeld University. doi: [10.4119/unibi/2946374](https://pub.uni-bielefeld.de/record/2946374).

**Pucker B.**, Pandey A., Weisshaar B. and Stracke R. (2020). The R2R3-MYB gene family in banana (Musa acuminata): genome-wide identification, classification and expression patterns. PLOS ONE 15(10): e0239275. doi:[10.1371/journal.pone.0239275](https://doi.org/10.1371/journal.pone.0239275).

**Pucker, B.**, Reiher, F. and Schilbert, H. S. Automatic identification of players in the flavonoid biosynthesis with application on the biomedicinal plant Croton tiglium. Plants 2020, 9, 1103. doi:[10.3390/plants9091103](https://www.mdpi.com/2223-7747/9/9/1103/htm).


Choi, J., Lee, T., Cho, J., Servante, E., **Pucker, B.**, Summers, W., et al. The negative regulator SMAX1 controls mycorrhizal symbiosis and strigolactone biosynthesis in rice. Nature Communications 11, 2114 (2020). doi:[10.1038/s41467-020-16021-1](https://www.nature.com/articles/s41467-020-16021-1).


Schilbert H. M., Rempel A. and **Pucker, B.** (2020). Comparison of read mapping and variant calling tools for the analysis of plant NGS data. Plants. doi:[10.3390/plants9040439](https://www.mdpi.com/2223-7747/9/4/439).


Frey K. & **Pucker, B.** (2020). Animal, fungi, and plant genome sequences harbour different non-canonical splice sites. Cells. doi:[10.3390/cells9020458](https://www.mdpi.com/2073-4409/9/2/458).


Holtgräwe, D., Rosleff Sörensen, T., Hausmann, L., **Pucker, B.**, Viehöver, P., Töpfer, R., Weisshaar, B. (2020). A Partially Phase-Separated Genome Sequence Assembly of the Vitis Rootstock 'Börner' (Vitis riparia x Vitis cinerea) and its Exploitation for Marker Development and Targeted Mapping. Front. Plant Sci. doi:[10.3389/fpls.2020.00156](https://doi.org/10.3389/fpls.2020.00156).


Frey, K.*, Hafner, A.* and **Pucker B.** (2020). The Reuse of Public Datasets in the Life Sciences: Potential Risks and Rewards.  PeerJ 8:e9954. doi: [10.7717/peerj.9954](https://doi.org/10.7717/peerj.9954)


Siadjeu C.+, **Pucker B.+**, Viehoever P., Albach D. and Weisshaar B. (2020). High contiguity de novo genome sequence assembly of Trifoliate yam (Dioscorea dumetorum) using long read sequencing. Genes. doi:[10.3390/genes11030274](https://www.mdpi.com/2073-4425/11/3/274/htm).

\+ authors contributed equally


Busche, M.+, **Pucker, B.+**, Viehoever, P., Weisshaar, B., & Stracke, R. (2019). Genome Sequencing of _Musa acuminata_ Dwarf Cavendish Reveals a Duplication of a Large Segment of Chromosome 2. G3: Genes|Genomes|Genetics. 2019.doi:[10.1534/g3.119.400847](https://doi.org/10.1534/g3.119.400847).

\+ authors contributed equally



## 2019

Holtgraewe, D., Rosleff Soerensen, T., Hausmann, L., **Pucker, B.**, Viehoever, P., Toepfer, R., Weisshaar, B. (2019). A Partially Phase-Separated Genome Sequence Assembly of the Vitis Rootstock 'Börner' (Vitis riparia x Vitis cinerea) and its Exploitation for Marker Development and Targeted Mapping. bioRxiv 854687. doi: [10.1101/854687](https://doi.org/10.1101/854687). 


**Pucker B.** & Schilbert H. Genomics and Transcriptomics Advance in Plant Sciences. Molecular Approaches in Plant Biology and Environmental Challenges. Springer. 2019. ISBN 978-981-15-0690-1. doi:[10.1007/978-981-15-0690-1](https://www.springer.com/gp/book/9789811506895).


Sheehan, H., Feng, T., Walker-Hale, N., Lopez-Nieves, S., **Pucker, B.**, Guo, R., Yim, W. C., Badgami, R., Timoneda, A., Zhao, L., Tiley, H., Copetti, D., Sanderson, M. J., Cushman, J. C., Moore, M. J., Smith, S. A., Brockington, S. F. Evolution of L-DOPA 4,5-dioxygenase activity allows for recurrent specialisation to betalain pigmentation in Caryophyllales. New Phytologist. 2019. doi:[10.1111/nph.16089](https://doi.org/10.1111/nph.16089).


**Pucker, B.**, Rückert, C., Stracke, R., Viehöver, P., Kalinowski, J., Weisshaar, B.. 25 years of propagation in suspension cell culture results in substantial alterations of the _Arabidopsis thaliana_ genome. bioRxiv. 2019.doi:[10.3390/genes10090671](https://www.mdpi.com/2073-4425/10/9/671).

Timoneda, A., Feng, T., Sheehan, H., Walker-Hale, N., **Pucker, B.**, Lopez-Nieves, S., Guo, R., Brockington, S. (2019). The evolution of betalain biosynthesis in Caryophyllales. New Phytologist. doi:[10.1111/nph.15980](https://doi.org/10.1111/nph.15980).

**Pucker, B.** Schilbert, H.M., Schumacher, S.F. (2019). Integrating Molecular Biology and Bioinformatics Education. Journal of Integrative Bioinformatics. doi:[10.1515/jib-2019-0005](https://doi.org/10.1515/jib-2019-0005).

**Pucker, B.**, Feng, T., Brockington, S. (2019). Next generation sequencing to investigate genomic diversity in Caryophyllales. bioRxiv 646133; doi:[10.1101/646133](https://www.biorxiv.org/content/10.1101/646133v2.full).

**Pucker, B.**, Holtgräwe, D., Stadermann, K. B., Frey, K., Hüttel, B., Reinhardt, R., & Weisshaar, B. (2019). A Chromosome-level Sequence Assembly Reveals the Structure of the _Arabidopsis thaliana_ Nd-1 Genome and its Gene Set. PLOS ONE:e0216233. doi:[10.1371/journal.pone.0216233](https://doi.org/10.1371/journal.pone.0216233).

Frey K., **Pucker, B.** (2019). Animal, fungi, and plant genome sequences harbour different non-canonical splice sites. bioRxiv 616565. doi:[10.1101/616565](https://www.biorxiv.org/content/10.1101/616565v2.full).

**Pucker, B.** (2019). Mapping-based genome size estimation. bioRxiv. doi:[10.1101/607390](https://www.biorxiv.org/content/10.1101/607390v1.full).

Baasner, J.-S., Howard, D., **Pucker, B.** (2019). Influence of neighboring small sequence variants on functional impact prediction. bioRxiv. doi:[10.1101/596718](https://www.biorxiv.org/content/10.1101/596718v2.full).

**Pucker, B.** (2019). _De novo_ Nd-1 genome assembly reveals genomic diversity of _Arabidopsis thaliana_ and facilitates genome-wide non-canonical splice site analyses across plant species. Dissertation. DOI:[10.4119/unibi/2935794](https://pub.uni-bielefeld.de/download/2935794/2935982/Pucker_Dissertation.pdf).

**Pucker, B.** and Brockington, S.F. (2018). Genome-wide analyses supported by RNA-Seq reveal non-canonical splice sites in plant genomes. BMC Genomics. 2018;19(1). doi:[10.1186/s12864-018-5360-z](https://doi.org/10.1186/s12864-018-5360-z).


## 2018

Schilbert, H.M., Pellegrinelli, V., Rodriguez-Cuenca, S., Vidal-Puig, A. & **Pucker, B.** (2018). Harnessing natural diversity to identify key amino acid residues in prolidase. bioRxiv 423475; doi:[10.1101/423475](https://doi.org/10.1101/423475).

Brinkrolf, C., Henke, N. A., Ochel, L., **Pucker, B.**, Kruse, O., and Lutter, P. (2018). Modeling and Simulating the Aerobic Carbon Metabolism of a Green Microalga Using Petri Nets and New Concepts of VANESA. Journal of Integrative Bioinformatics 15. doi:[10.1515/jib-2018-0018](https://doi.org/10.1515/jib-2018-0018).

Whitford, C. M., Dymek, S., Kerkhoff, D., März, C., Schmidt, O., Edich, M., Droste, J., **Pucker, B.**, Rückert, C. and Kalinowski, J. (2018). Auxotrophy to Xeno-DNA: an exploration of combinatorial mechanisms for a high-fidelity biosafety system for synthetic biology applications. Journal of Biological Engineering, 12(1). doi:[10.1186/s13036-018-0105-8](https://rdcu.be/4u5u).

Haak, M., Vinke, S., Keller, W., Droste, J., Rückert, C., Kalinowski, J., & **Pucker, B.** (2018). High Quality _de novo_ Transcriptome Assembly of _Croton tiglium_. Frontiers in Molecular Biosciences, 5. doi:[10.3389/fmolb.2018.00062](https://doi.org/10.3389/fmolb.2018.00062).

Friedrich, A. and **Pucker, B.** (2018). Peer-review as a teaching method. Working Paper der AG Forschendes Lernen in der dghd, 2, 2018. Carl von Ossietzky Universität Oldenburg. [full text](https://pub.uni-bielefeld.de/download/2920633/2920815)


## 2017

**Pucker, B.**, Holtgräwe, D., & Weisshaar, B. (2017). Consideration of non-canonical splice sites improves gene prediction on the _Arabidopsis thaliana_ Niederzenz-1 genome sequence. BMC Research Notes, 10, 667. doi:[10.1186/s13104-017-2985-y](https://doi.org/10.1186/s13104-017-2985-y).


Alkhateeb, R., Rückert, C., Rupp, O., **Pucker, B.**, Hublik, G., Wibberg, D., Niehaus, K., Pühler, A., Vorhölter, F.J. (2017). Refined annotation of the complete genome of the phytopathogenic and xanthan producing _Xanthomonas campestris_ pv. campestris strain B100 based on RNA sequence data. Journal of Biotechnology. doi:[10.1016/j.jbiotec.2017.05.009](https://doi.org/10.1016/j.jbiotec.2017.05.009).


## 2016

**Pucker, B.**, Holtgräwe, D., Rosleff Sörensen, T., Stracke, R., Viehöver, P., and Weisshaar, B. (2016). A _de novo_ Genome Sequence Assembly of the _Arabidopsis thaliana_ Accession Niederzenz-1 Displays Presence/Absence Variation and Strong Synteny. PloS-ONE 11:e0164321. doi:[10.1371/journal.pone.0164321](https://doi.org/10.1371/journal.pone.0164321).


## 2014

Fust A., Hollmann B., **Pucker, B.**, Wollborn D., Tiemann J., Droste J., Brosda S., Blunk S., Riedl S., Bleckwehl T. (2014). The Transformers. From Carbon Dioxide to Biofuel. doi:[10.13140/RG.2.2.30504.98566](https://www.researchgate.net/publication/287330796_iGEM_Bielefeld-CeBiTec_2014).

Stracke, R., Holtgräwe, D., Schneider, J., **Pucker, B.**, Sörensen, T.R., and Weisshaar, B. (2014). Genome-wide identification and characterisation of R2R3-MYB genes in sugar beet (Beta vulgaris). BMC Plant Biol. 14: 249. doi:[10.1186/s12870-014-0249-8](https://doi.org/10.1186/s12870-014-0249-8).
