<p>
<a href="https://www.cebitec.uni-bielefeld.de/~bpucker#publications"><b>Publications</b></a> 
- <a href="https://www.cebitec.uni-bielefeld.de/~bpucker#funding"><b>Funding</b></a>
- <a href="https://www.cebitec.uni-bielefeld.de/~bpucker#research_interests"><b>Research Interests</b></a>
- <a href="https://www.cebitec.uni-bielefeld.de/~bpucker#CV"><b>CV</b></a>
</p>



<h1>Dr. Boas Pucker</h1>

<ul>
<li>groups:<br>
<a href="https://www.genomforschung.uni-bielefeld.de/en/">Genetics & Genomics of Plants</a> (Bielefeld University, Germany)<br>
<a href="https://www.ruhr-uni-bochum.de/mgpp/Seiten_en/index_e.html">Molecular Genetics and Physiology of Plants</a> (Ruhr University Bochum, Germany)<br>
<a href="https://www.plantsci.cam.ac.uk/research/sambrockington">Evolution and Diversity</a> (Department of Plant Sciences, University of Cambridge, UK)
</li>
<li>email: bpucker{a}cebitec{dot}uni-bielefeld.de, boas.pucker{a}rub.de, bp423{a}cam.ac{dot}uk</li>
<li>Google Scholar: <a href="https://scholar.google.de/citations?user=GnnyChAAAAAJ&hl=de&oi=ao">Boas Pucker</a></li>
<li>Publons: <a href="https://publons.com/researcher/1008011/boas-pucker/">Boas Pucker</a></li>
<li>ORCID: <a href="https://orcid.org/0000-0002-3321-7471">0000-0002-3321-7471</a></li>
<li>Twitter: <a href="https://twitter.com/boas_pucker">@boas_pucker</a>
<a href="https://twitter.com/boas_pucker?ref_src=twsrc%5Etfw" class="twitter-follow-button" data-lang="en" data-show-count="false">Follow @boas_pucker</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
</li>
<li>Github: <a href="https://github.com/bpucker">bpucker</a></li>
<li>LinkedIn: <a href="https://www.linkedin.com/in/boas-pucker-80aaa411b/">Boas Pucker</a></li>
</ul>

<p style="color:green"><b>Collaborations</b>: I am always interested in collaborations involving genomics or bioinformatic analyses. Please send me an email.</p>
<p style="color:green"><b>Reviews</b>: Review requests from high quality open access journals are welcome if they match 
<a href="https://www.cebitec.uni-bielefeld.de/~bpucker#research_interests" style="color:green">my expertise</a>.
</p>

<h3 id="DFGproject">Current project (DFG fellowship)</h3>
<p>Investigation of the <a href="https://gepris.dfg.de/gepris/projekt/436841671?language=en"><b>betalain pathway evolution</b></a> with Dr. Samuel Brockington
at the Department of Plant Sciences (University of Cambridge, UK): 
<a href="https://www.plantsci.cam.ac.uk/directory/pucker-boas">Boas Pucker (Plant Sciences)</font></a>
</p>


<h3 id="publications">Publications</h3>
<ul>


<li>Naik, J., Rajput, R., <b>Pucker, B.</b>, Stracke, R. and Pandey, A. 
The R2R3-MYB transcription factor MtMYB134 orchestrates flavonol biosynthesis in Medicago truncatula. 
Plant Mol Biol (2021). doi:10.1007/s11103-021-01135-x<br>
<a href="https://doi.org/10.1007/s11103-021-01135-x">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://link.springer.com/content/pdf/10.1007/s11103-021-01135-x.pdf">PDF</a>
</li>



<li><b>Pucker, B.</b>*, Kleinb&ouml;lting, N.* and Weisshaar, B.  (2021).  
Large scale genomic rearrangements in selected Arabidopsis thaliana T-DNA lines are caused by T-DNA insertion mutagenesis. doi: 10.1101/2021.03.03.433755<br>
<a href="https://www.biorxiv.org/content/10.1101/2021.03.03.433755v2">full text</a>
<a href="https://www.biorxiv.org/content/10.1101/2021.03.03.433755v2.full.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2952434">PUB</a>
<a href="https://github.com/bpucker/GKseq">github</a>
<a href="https://github.com/nkleinbo/loreta">github</a>
<br>
*shared first authorship
</li>


<li>Sielemann, K., Weisshaar, B. and <b>Pucker, B.</b> (2021).  
Reference-based QUantification Of gene Dispensability (QUOD). Plant Methods 17, 18. doi: 10.1186/s13007-021-00718-5<br>
<a href="https://doi.org/10.1186/s13007-021-00718-5">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://doi.org/10.1101/2020.04.28.065714">bioRxiv</a>
<a href="https://plantmethods.biomedcentral.com/track/pdf/10.1186/s13007-021-00718-5.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2943037">PUB</a>
<a href="https://github.com/KatharinaFrey/QUOD">github</a>
</li>

<li>
Xu B., Taylor L., <b>Pucker B.</b>, Feng T., Glover B. J. and Brockington S. (2021).
The land plant-specific MIXTA-MYB lineage is implicated in the early evolution of the plant cuticle and the colonization of land. New Phytologist. doi:10.1111/nph.16997.<br>
<a href="https://doi.org/10.1111/nph.16997">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://pub.uni-bielefeld.de/record/2946565">PUB</a>
</li>


<li>
<b>Pucker B.</b>, Schwandner A., Becker S., Hausmann L., Vieh&ouml;ver P., T&ouml;pfer R., Weisshaar B., Holtgr&auml;we D. (2020).
RNA-Seq Time Series of Vitis vinifera Bud Development Reveals Correlation of Expression Patterns with the Local Temperature Profile.
 Plants 2020, 9(11), 1548; doi:10.3390/plants9111548.<br>
 <a href="https://www.mdpi.com/2223-7747/9/11/1548/htm">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://doi.org/10.1101/2020.10.18.344176">preprint</a>
<a href="https://www.mdpi.com/2223-7747/9/11/1548/pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2946639">PUB</a>
</li>


<li>
<b>Pucker B.</b> and Brockington S. (2020). Pitaya transcriptome assemblies and investigation of transcript abundances. Bielefeld University. doi: 10.4119/unibi/2946374.<br>
<a href="https://pub.uni-bielefeld.de/record/2946374">PUB</a>
<a href="https://github.com/bpucker/pitaya">github</a>
</li>


<li><b>Pucker B.</b>, Pandey A., Weisshaar B. and Stracke R. (2020).  
The <i>R2R3-MYB</i> gene family in banana (Musa acuminata): genome-wide identification, classification and expression patterns.
 PLOS ONE 15(10): e0239275. doi:10.1371/journal.pone.0239275<br>
<a href="https://doi.org/10.1371/journal.pone.0239275">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://doi.org/10.1101/2020.02.03.932046">preprint</a>
<a href="https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0239275&type=printable">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2940694">PUB</a>
<a href="https://github.com/bpucker/bananaMYB">github</a>
</li>


<li>Sielemann, K.*, Hafner, A.* and <b>Pucker B.</b> (2020).  
The Reuse of Public Datasets in the Life Sciences: Potential Risks and Rewards.  PeerJ 8:e9954. doi: 10.7717/peerj.9954<br>
<a href="https://doi.org/10.7717/peerj.9954">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://peerj.com/articles/9954.pdf">PDF</a>
<a href="https://www.preprints.org/manuscript/202002.0146/v2">preprint</a>
<a href="https://peerj.com/articles/9954.pdf">PUB</a>
*shared first authorship
</li>


<li><b>Pucker, B.</b>,  Reiher, F. and Schilbert, H. M.   
Automatic identification of players in the flavonoid biosynthesis with application on the biomedicinal plant <i>Croton tiglium</i>. 
Plants 2020, 9, 1103. doi:10.3390/plants9091103.<br>
<a href="https://www.mdpi.com/2223-7747/9/9/1103/htm">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://www.mdpi.com/2223-7747/9/9/1103/pdf">PDF</a>
<a href="https://doi.org/10.1101/2020.06.27.175067">bioRxiv</a>
<a href="https://pub.uni-bielefeld.de/record/2944597">PUB</a>
<a href="https://github.com/bpucker/KIPEs">github</a>
</li>


<li>
Pellegrinelli V., Rodriguez-Cuenca S., Rouault C., Schilbert H., Virtue S., Moreno-Navarrete J. M.,
Bidault G., del Carmen Vazquez Borrego M., Dias A. R., <b>Pucker B.</b>,
Dale M., Campbell M., Carobbio S., Aron-Wisnewsky J., Mora S., Masiero M., Emmanouilidou A.,
Mukhopadhyay S., Dougan G., den Hoed M., Loos R., Fernandez-Real J. M., Chiarugi D., Clement K. and Vidal-Puig T. 
Dysregulation of macrophage PEPD in obesity determines adipose tissue fibro-inflammation and insulin resistance.
doi:10.21203/rs.3.rs-57182.<br>
<a href="https://doi.org/10.21203/rs.3.rs-57182/v1">full text</a>
<a href="https://www.researchsquare.com/article/rs-57182/v1.pdf">PDF</a>
</li>


<li>Choi, J., Lee, T., Cho, J., Servante, E., <b>Pucker, B.</b>, Summers, W., et al.   
The negative regulator SMAX1 controls mycorrhizal symbiosis and strigolactone biosynthesis in rice. 
Nature Communications 11, 2114 (2020). doi:10.1038/s41467-020-16021-1.<br>
<a href="https://www.nature.com/articles/s41467-020-16021-1">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://www.repository.cam.ac.uk/handle/1810/304408">Apollo (free full text)</font></a>
<a href="https://pub.uni-bielefeld.de/record/2943062">PUB</a>
</li>


<li>Schilbert H. M., Rempel A. and <b>Pucker, B.</b> (2020).  
Comparison of read mapping and variant calling tools for the analysis of plant NGS data. Plants. doi:10.3390/plants9040439.<br>
<a href="https://www.mdpi.com/2223-7747/9/4/439/htm">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://doi.org/10.1101/2020.03.10.986059">bioRxiv</a>
<a href="https://www.biorxiv.org/content/10.1101/2020.03.10.986059v2.full.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2942341">PUB</a>
<a href="https://doi.org/10.4119/unibi/2942069">data</a>
</li>


<li>Siadjeu C.*, <b>Pucker B.*</b>, Viehoever P., Albach D. and Weisshaar B. (2020).  
High contiguity de novo genome sequence assembly of Trifoliate yam (Dioscorea dumetorum) using long read sequencing.
Genes. doi:10.3390/genes11030274.<br>
<a href="https://www.mdpi.com/2073-4425/11/3/274/htm">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://doi.org/10.1101/2020.01.31.928630">bioRxiv</a>
<a href="https://www.biorxiv.org/content/10.1101/2020.01.31.928630v3.full.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2940688">PUB</a>
<a href="https://github.com/bpucker/yam">github</a>
<a href="https://dx.doi.org/10.17504/protocols.io.bcvyiw7w">DNA extraction protocol</a>
<a href="https://doi.org/10.4119/unibi/2941469">data (PUB)</a>
<br>
*shared first authorship
</li>

<li>Frey K. & <b>Pucker, B.</b> (2020).  
Animal, fungi, and plant genome sequences harbour different non-canonical splice sites. Cells. doi:10.3390/cells9020458.<br>
<a href="https://www.mdpi.com/2073-4409/9/2/458">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://www.mdpi.com/2073-4409/9/2/458/pdf">PDF</a>
<a href="https://doi.org/10.1101/616565">bioRxiv</a>
<a href="https://pub.uni-bielefeld.de/record/2935311">PUB</a>
<a href="https://github.com/bpucker/ncss2018">github</a>
<a href="https://pub.uni-bielefeld.de/record/2934220">data (fungi)</a>
<a href="https://pub.uni-bielefeld.de/record/2934226">data (animals)</a>
</li>

<li>
Holtgr&auml;we, D., Rosleff S&ouml;rensen, T., Hausmann, L., <b>Pucker, B.</b>, Vieh&ouml;ver, P., T&ouml;pfer, R., Weisshaar, B. (2020).
A Partially Phase-Separated Genome Sequence Assembly of the Vitis Rootstock 'B&ouml;rner' 
(<i>Vitis riparia</i> x <i>Vitis cinerea</i>) and its Exploitation for Marker Development and Targeted Mapping. 
Front. Plant Sci. doi:10.3389/fpls.2020.00156.<br>
<a href="https://doi.org/10.3389/fpls.2020.00156">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/25/854687.full-text.pdf">PDF</a>
<a href="https://www.biorxiv.org/content/10.1101/854687v1">bioRxiv</a>
<a href="https://pub.uni-bielefeld.de/record/2939244">PUB</a>
<a href="http://doi.org/10.4119/unibi/2938185">data set 1</a>
<a href="http://doi.org/10.4119/unibi/2938180">data set 2</a>
<a href="https://doi.org/10.4119/unibi/2938178">data set 3</a>
</li>


<li> Busche M*, <b>Pucker B*</b>, Vieh&ouml;ver P, Weisshaar B, Stracke R. (2020).
Genome Sequencing of Musa acuminata Dwarf Cavendish Reveals a Duplication of a Large Segment of Chromosome 2. 
G3: Genes|Genomes|Genetics. doi:10.1534/g3.119.400847.<br>
<a href="https://doi.org/10.1534/g3.119.400847">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/691923.full-text.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2938807">PUB</a>
<a href="https://www.biorxiv.org/content/10.1101/691923v4.full">bioRxiv</a>
<a href="https://github.com/bpucker/banana">github</a>
<a href="https://pub.uni-bielefeld.de/record/2937697">genome assemblies</a>
<a href="https://pub.uni-bielefeld.de/record/2936278">sequence variants</a>
<br>
*shared first authorship
</li>


<li>
Sheehan, H., Feng, T., Walker-Hale, N., Lopez-Nieves, S., <b>Pucker, B.</b>, Guo, R., Yim, W. C., Badgami, R., Timoneda, A., Zhao, L., Tiley, H., Copetti, D., Sanderson, M. J., Cushman, J. C., Moore, M. J., Smith, S. A., Brockington, S. F. 
Evolution of L-DOPA 4,5-dioxygenase activity allows for recurrent specialisation to betalain pigmentation in Caryophyllales. 
New Phytologist. 2020. doi:10.1111/nph.16089<br>
<a href="https://doi.org/10.1111/nph.16089">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://nph.onlinelibrary.wiley.com/doi/epdf/10.1111/nph.16089">PDF</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/31369159">PubMed</a>
<a href="https://pub.uni-bielefeld.de/record/2936783">PUB</a>
<a href="https://www.repository.cam.ac.uk/handle/1810/295044">Apollo</a>
</li>


<li>
<b>Pucker B</b> & Schilbert H.
Genomics and Transcriptomics Advance in Plant Sciences. 
Molecular Approaches in Plant Biology and Environmental Challenges. 
Springer. 2019. ISBN 978-981-15-0690-1. doi:10.1007/978-981-15-0690-1<br>
<a href="https://www.springer.com/gp/book/9789811506895">springer<font color="blue">(peer-reviewed book chapter)</font></a>
<a href="https://pub.uni-bielefeld.de/record/2938013">PUB</a>
</li>

<li>
<b>Pucker B</b>, R&uuml;ckert C, Stracke R, Vieh&ouml;ver P, Kalinowski J, Weisshaar B. 
Twenty-Five Years of Propagation in Suspension Cell Culture Results in Substantial Alterations of the Arabidopsis Thaliana Genome. 
Genes. 2019. doi:10.3390/genes10090671<br>
<a href="https://www.mdpi.com/2073-4425/10/9/671">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://www.mdpi.com/2073-4425/10/9/671/pdf">PDF</a>
<a href="https://www.biorxiv.org/content/10.1101/710624v3.full">bioRxiv</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/31480756">PubMed</a>
<a href="https://pub.uni-bielefeld.de/record/2936644">PUB</a>
<a href="https://github.com/bpucker/At7">github</a>
</li>


<li>
Timoneda, A., Feng, T., Sheehan, H., Walker-Hale, N., <b>Pucker, B.</b>, Lopez-Nieves, S., Guo, R., Brockington, S. (2019).
The evolution of betalain biosynthesis in Caryophyllales.
New Phytologist. doi:10.1111/nph.15980<br>
<a href="https://doi.org/10.1111/nph.15980">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://nph.onlinelibrary.wiley.com/doi/epdf/10.1111/nph.15980">PDF</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/31172524">PubMed</a>
<a href="https://pub.uni-bielefeld.de/record/2936079">PUB</a>
</li>

<li>
 <b>Pucker, B.</b> Schilbert, H.M., Schumacher, S.F. (2019). 
Integrating Molecular Biology and Bioinformatics Education. 
Journal of Integrative Bioinformatics. doi:10.1515/jib-2019-0005.<br>
<a href="https://doi.org/10.1515/jib-2019-0005">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://www.degruyter.com/downloadpdf/j/jib.2019.16.issue-3/jib-2019-0005/jib-2019-0005.xml">PDF</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/31145692">PubMed</a>
<a href="https://www.preprints.org/manuscript/201811.0183/v2">preprint</a>
<a href="https://pub.uni-bielefeld.de/record/2935844">PUB</a><br>
<a href="https://github.com/bpucker/AppliedGenomeResearch">Applied Genome Research (github)</a>
<a href="https://github.com/bpucker/MolecularMethodsInGenomeResearch">Molecular Methods in Genome Research (github)</a>
</li>

<li><b>Pucker, B.</b>, Feng, T., Brockington, S. (2019). 
Next generation sequencing to investigate genomic diversity in Caryophyllales.
bioRxiv 646133; doi:10.1101/646133.
<br>
<a href="https://www.biorxiv.org/content/10.1101/646133v3.full">full text</a>
<a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/13/646133.full.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2935737">PUB</a>
<a href="https://docs.cebitec.uni-bielefeld.de/s/pZ4kGpPEDtTPgjW">data</a>
<a href="https://github.com/bpucker/GenomeAssemblies2018">github</a>
</li>

<li><b>Pucker, B.</b>, Holtgraewe, D., Stadermann, K. B., Frey, K., Huettel, B., Reinhardt, R., & Weisshaar, B. (2019). 
A Chromosome-level Sequence Assembly Reveals the Structure of the <i>Arabidopsis thaliana</i> Nd-1 Genome and its Gene Set. PLOS ONE: e0216233. doi: 10.1371/journal.pone.0216233<br>
<a href="https://doi.org/10.1371/journal.pone.0216233">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0216233&type=printable">PDF</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/31112551">PubMed</a>
<a href="https://www.biorxiv.org/content/10.1101/407627v3.full">bioRxiv</a>
<a href="https://pub.uni-bielefeld.de/record/2930926">PUB</a>
<a href="https://doi.org/10.5447/IPK/2019/4">data</a>
<a href="https://github.com/bpucker/Nd1_PacBio">github</a>
</li>

<li><b>Pucker, B.</b> (2019).  
Mapping-based genome size estimation. bioRxiv. doi:10.1101/607390.<br>
<a href="https://www.biorxiv.org/content/10.1101/607390v1.full">full text</a>
<a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/607390.full-text.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2934975">PUB</a>
<a href="https://github.com/bpucker/MGSE">github (MGSE)</a>
</li>

<li>Baasner, J.-S., Howard, D., <b>Pucker, B.</b> (2019).  
Influence of neighboring small sequence variants on functional impact prediction. bioRxiv. doi:10.1101/596718.<br>
<a href="https://www.biorxiv.org/content/10.1101/596718v2.full">full text</a>
<a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/27/596718.full-text.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2934778">PUB</a>
<a href="https://github.com/bpucker/NAVIP">github (NAVIP)</a>
<a href="https://github.com/bpucker/variant_calling">github (other scripts)</a>
</li>

<li><b>Pucker, B.</b> (2019). <i>De novo</i> Nd-1 genome assembly reveals genomic diversity of <i>Arabidopsis thaliana</i> and facilitates genome-wide non-canonical splice site analyses across plant species. Dissertation. doi:10.4119/unibi/2935794.<br>
<a href="https://pub.uni-bielefeld.de/download/2935794/2935982/Pucker_Dissertation.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2935794">PUB</a>
<a href="http://doi.org/10.4119/unibi/2935794">DOI</a>
</li>

<li>
 <b>Pucker, B.</b> and Brockington, S.F. (2018). 
Genome-wide analyses supported by RNA-Seq reveal non-canonical splice sites in plant genomes. 
BMC Genomics. 2018;19(1). doi:10.1186/s12864-018-5360-z.<br>
<a href="https://doi.org/10.1186/s12864-018-5360-z">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://bmcgenomics.biomedcentral.com/track/pdf/10.1186/s12864-018-5360-z">PDF</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30594132">PubMed</a>
<a href="https://doi.org/10.1101/428318">bioRxiv</a>
<a href="https://pub.uni-bielefeld.de/record/2932870">PUB</a>
<a href="https://doi.org/10.17863/CAM.34763">Apollo</a>
<a href="https://pub.uni-bielefeld.de/record/2931315">data</a>
<a href="https://github.com/bpucker/ncss2018">github</a>
</li>

<li>
Schilbert, H.M., Pellegrinelli, V., Rodriguez-Cuenca, S., Vidal-Puig, A. & <b>Pucker, B.</b> (2018). 
Harnessing natural diversity to identify key amino acid residues in prolidase. 
bioRxiv 423475; doi:10.1101/423475.<br>
<a href="https://doi.org/10.1101/423475">full text</a>
<a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/24/423475.full-text.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2931225">PUB</a>
<a href="https://github.com/bpucker/PEPD">github</a>
<a href="https://doi.org/10.5281/zenodo.2573170">zenodo</a>
</li>

<li>Brinkrolf, C., Henke, N. A., Ochel, L., <b>Pucker, B.</b>, Kruse, O., and Lutter, P. (2018).
Modeling and Simulating the Aerobic Carbon Metabolism of a Green Microalga Using Petri Nets and New Concepts of VANESA.
Journal of Integrative Bioinformatics 15. doi:10.1515/jib-2018-0018.<br>
<a href="https://doi.org/10.1515/jib-2018-0018">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://www.degruyter.com/downloadpdf/j/jib.2018.15.issue-3/jib-2018-0018/jib-2018-0018.xml">PDF</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30218605">PubMed</a>
<a href="https://pub.uni-bielefeld.de/record/2931064">PUB</a>
<a href="http://agbi.techfak.uni-bielefeld.de/vanesa">VANESA</a>
</li>

<li>Whitford, C. M., Dymek, S., Kerkhoff, D., M&auml;rz, C., Schmidt, O., Edich, M., Droste, J., <b>Pucker, B.</b>, R&uuml;ckert, C. and Kalinowski, J. (2018). 
Auxotrophy to Xeno-DNA: an exploration of combinatorial mechanisms for a high-fidelity biosafety system for synthetic biology applications. 
Journal of Biological Engineering, 12(1). doi:10.1186/s13036-018-0105-8.<br>
<a href="https://rdcu.be/4u5u">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://jbioleng.biomedcentral.com/track/pdf/10.1186/s13036-018-0105-8">PDF</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30123321">PubMed</a>
<a href="https://pub.uni-bielefeld.de/record/2930296">PUB</a>
<a href="https://doi.org/10.17863/CAM.26216">Apollo</a>
<a href="https://igem.org/Main_Page">iGEM</a>
</li>

<li>Haak, M., Vinke, S., Keller, W., Droste, J., R&uuml;ckert, C., Kalinowski, J., & <b>Pucker, B.</b> (2018).
High Quality <i>de novo</i> Transcriptome Assembly of <i>Croton tiglium</i>.
Frontiers in Molecular Biosciences, 5. doi:https://doi.org/10.3389/fmolb.2018.00062.<br>
<a href="https://doi.org/10.3389/fmolb.2018.00062">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6041412/pdf/fmolb-05-00062.pdf">PDF</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30027092">PubMed</a>
<a href="https://pub.uni-bielefeld.de/record/2921026">PUB</a>
<a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA416498">RNA-Seq data</a>
<a href="https://www.ncbi.nlm.nih.gov/nuccore/GGDV00000000.1">assembly</a>
</li>

<li>Friedrich, A., & <b>Pucker, B.</b> (2018). 
Peer-review as a teaching method. Working Paper der AG Forschendes Lernen in der dghd, 2, 2018. 
Carl von Ossietzky Universit&auml;t Oldenburg. <br>
<a href="https://pub.uni-bielefeld.de/download/2920633/2920815">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://pub.uni-bielefeld.de/download/2920633/2920815">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2920633">PUB</a>
<a href="https://github.com/bpucker/APPLS">github</a>
</li>

<li><b>Pucker, B.</b>, Holtgr&auml;we, D., & Weisshaar, B. (2017). 
Consideration of non-canonical splice sites improves gene prediction on the <i>Arabidopsis thaliana</i> Niederzenz-1 genome sequence.
BMC Research Notes, 10, 667. doi:10.1186/s13104-017-2985-y.<br>
<a href="https://doi.org/10.1186/s13104-017-2985-y">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://bmcresnotes.biomedcentral.com/track/pdf/10.1186/s13104-017-2985-y">PDF</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/29202864">PubMed</a>
<a href="https://pub.uni-bielefeld.de/record/2915524">PUB</a>
</li>

<li>Karsten, L., Bergen, D., Drake, C., Dymek, S., Edich, M., Haak, M., Kerkhoff, D., Kerkhoff, Y., Liebers, M., M&auml;rz, C., Schl&uuml;ter, L., Schmidt, O., Vinke, S., Whitford, C.M., <b>Pucker, B.</b>, Droste, J., R&uuml;ckert, C., M&uuml;ller, K.M., Kalinowski, J. (2017). 
Expanding The Genetic Code. Bielefeld University. doi:10.13140/RG.2.2.20342.91203.<br>
<a href="https://www.researchgate.net/publication/321586905_Expanding_The_Genetic_Code?channel=doi&linkId=5a27ea640f7e9b71dd0cca08&showFulltext=true">full text</a>
<a href="https://pub.uni-bielefeld.de/record/2930821">PUB</a>
</li>

<li>Alkhateeb, R., R&uuml;ckert, C., Rupp, O., <b>Pucker, B.</b>, Hublik, G., Wibberg, D., Niehaus, K., P&uuml;hler, A., Vorh&ouml;lter, F.J. (2017). 
Refined annotation of the complete genome of the phytopathogenic and xanthan producing <i>Xanthomonas campestris</i> pv. campestris strain B100 based on RNA sequence data.
Journal of Biotechnology. doi:10.1016/j.jbiotec.2017.05.009.<br>
<a href="https://doi.org/10.1016/j.jbiotec.2017.05.009">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/28506932">PubMed</a>
<a href="https://pub.uni-bielefeld.de/record/2911462">PUB</a>
</li>

<li><b>Pucker, B.</b>, Holtgr&auml;we, D., Rosleff S&ouml;rensen, T., Stracke, R., Vieh&ouml;ver, P., and Weisshaar, B. (2016).
A <i>de novo</i> Genome Sequence Assembly of the <i>Arabidopsis thaliana</i> Accession Niederzenz-1 Displays Presence/Absence Variation and Strong Synteny.
PloS-ONE 11:e0164321. doi:10.1371/journal.pone.0164321.<br>
<a href="https://doi.org/10.1371/journal.pone.0164321">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0164321&type=printable">PDF</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/27711162">PubMed</a>
<a href="https://pub.uni-bielefeld.de/record/2905844">PUB</a>
<a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA302255">NCBI BioProject</a>
</li>

<li>Frommer, B, Hain, C., Kampa, J., K&ouml;nigs, C., Linder, M., Perez Knoche, S., Sahin, M., Schmidt, P., Sch&ouml;ller, M., Kalinowski, J., M&uuml;ller, K.M., Droste, J., Hoffmann, N., Roeloffs, F., <b>Pucker, B.</b> (2016).
Generating Binding Proteins - via <i>in vivo</i> directed evolution in <i>E. coli</i>. doi:10.13140/RG.2.2.23164.95367.<br>
<a href="https://www.researchgate.net/publication/310965263_Generating_Binding_Proteins_-_via_in_vivo_directed_evolution_in_E_coli">full text</a>
</li>

<li>Fust A., Hollmann B., <b>Pucker, B.</b>, Wollborn D., Tiemann J., Droste J., Brosda S., Blunk S., Riedl S., Bleckwehl T. (2014).
The Transformers. From Carbon Dioxide to Biofuel. doi:10.13140/RG.2.2.30504.98566.<br>
<a href="https://www.researchgate.net/publication/287330796_iGEM_Bielefeld-CeBiTec_2014">full text</a>
<a href="https://pub.uni-bielefeld.de/record/2905012">PUB</a>
<a href="https://blogs.plos.org/collections/igem-report-006/">PLOS Report</a>
</li>

<li>Stracke, R., Holtgr&auml;we, D., Schneider, J., <b>Pucker, B.</b>, S&ouml;rensen, T.R., and Weisshaar, B. (2014).
Genome-wide identification and characterisation of R2R3-MYB genes in sugar beet (<i>Beta vulgaris</i>).
BMC Plant Biol. 14: 249. doi:10.1186/s12870-014-0249-8.<br>
<a href="https://doi.org/10.1186/s12870-014-0249-8">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://bmcplantbiol.biomedcentral.com/track/pdf/10.1186/s12870-014-0249-8">PDF</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/25249410">PubMed</a>
<a href="https://pub.uni-bielefeld.de/record/2696778">PUB</a>
<a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA254489">RNA-Seq data</a>
</li>

</ul>


<br> <br>


<h3 id="funding">Awards, funding, and scholarships</h3>
<ul>
<li>2019 <a href="https://www.dfg.de/">DFG</a> Research Fellowship for 'Understanding the convergent evolution of a complex metabolic trait, the Betalains, through comparative genomics and co-expression networks'
in <a href="https://www.plantsci.cam.ac.uk/research/sambrockington">Evolution and Diversity</a>, Department of Plant Sciences, University of Cambridge, UK
</li>
<li>2019 Research Fellowship for Plant Genomics and Bioinformatics</li>
<li>2018 <a href="https://www.stifterverband.org/digital-lehrfellows-nrw">Fellowship for Digital Innovations in Academic Teaching</a> 
(<a href="https://www.stifterverband.org/digital-lehrfellows/2017/pucker">project description in German</a>)
</li>
<li>2018 <a href="https://www.daad.de/en/">DAAD scholarship</a></li>
<li>2015 Award for excellent master's thesis (<a href="https://www.im.nrw/friedrich-wilhelm-helweg-stiftung">Friedrich Wilhelm Helweg Foundation</a>)</li>
<li>2014 <a href="https://www.deutschlandstipendium.de/de/english-1700.html">Deutschlandstipendium</a></li>
</ul>


<h3 id="research_interests">Research Interests</h3>

<h4 id="genome_research">Genome Research</h4>
<ul>
<li>high molecular weigth DNA extraction and long read sequencing (Oxford Nanopore Technologies, ONT)</li>
<li><i>de novo</i> genome assembly and annotation</li>
<li>comparative genomics and pangenomics</li>
<li>read mapping, variant calling (SNVs, InDels, SVs), and variant annotation</li>
<li>genome size and transposable elements</li>
</ul>


<br>


<h4 id="rna_seq">Transcriptomics / RNA-Seq</h4>
<ul>
<li><i>De novo</i> transcriptome assembly</li>
<li>Non-canonical splice sites</li>
<li>Differential gene expression analyses</li>
<li>Flowering time control and berry ripening in <i>Vitis vinifera</i></li>
</ul>

<br>


<h4 id="evolution">Evolution</h4>
<ul>
<li>Evolution of the <a href="https://doi.org/10.1111/nph.15980"><b>betalain</b></a> and <b>flavonoid</b> biosynthesis</li>
<li>Genome evolution</li>
<li>Evolution of the <i>R2R3-MYB</i> transcription factor gene family (<i>Beta vulgaris</i> and <i>Musa acuminata</i>)</li>
</ul>

<br>

<h4 id="plantscpecies">Studied Plant Species</h4>
<p>
	<i>Arabidopsis thaliana</i>, 
	<i>Arabidopsis halleri</i>, 
	<i>Beta vulgaris</i>, 
	<i>Beta coroliflora</i>, 
	<i>Beta lomatogona</i>, 
	<i>Brassica napus</i>, 
	<i>Cicer arietinum</i>, 
	<i>Croton tiglium</i>, 
	<i>Dioscorea dumetorum</i>, 
	<i>Kewa caespitosa</i>, 
	<i>Macarthuria australis</i>, 
	<i>Medicago truncatula</i>, 
	<i>Musa acuminata</i>, 
	<i>Pharnaceum exiguum</i>,
	<i>Simmondsia chinensis</i>, 
	<i>Vitis vinifera</i>
</p>
<p>
various other plant species especially members of the Caryophyllales
</p>
<br>


<h4 id="tools">Developed Bioinformatic Tools</h4>
<ul>
<li>
<b>KIPEs</b> (<u>K</u>nowledge-based <u>I</u>dentification of <u>P</u>athway <u>E</u>nzymes) identifieds all candidates of a metabolic pathway (e.g. flavonoid biosynthesis) or all members of a gene family (e.g. R2R3 MYBs).
Read the <a href="https://www.mdpi.com/2223-7747/9/9/1103/htm">publication</a> or download KIPEs from <a href="https://github.com/bpucker/KIPEs">github</a>.
</li>
<li>
<b>MGSE</b> (<u>M</u>apping-based <u>G</u>enome <u>S</u>ize <u>E</u>stimator) utilizes the mapping of short reads to a long read genome assembly to estimate the genome size. Read the 
<a href="https://doi.org/10.1101/607390"> preprint</a> or download MGSE from <a href="https://doi.org/10.1101/607390">github</a>.
</li>
<li>
<b>NAVIP</b> (<u>N</u>eighborhood-<u>A</u>ware <u>V</u>ariant <u>I</u>mpact <u>P</u>redictor) predicts the functional impact of sequence variants by considering all variants in a coding sequences simultaneously.
Read the <a href="https://www.biorxiv.org/content/10.1101/596718v2.full">preprint</a> or download NAVIP from <a href="https://github.com/bpucker/NAVIP">github</a>.
</li>
<li>
Most of my scripts are publicly available at github: <a href="https://github.com/bpucker">bpucker</a>.
</li>

</ul>


<br>



<h3 id="teaching">Teaching</h3>
<h4 id="courses">Courses</h4>
<p>The <a href="https://ekvv.uni-bielefeld.de/kvv_publ/publ/Lehrende_Veranstaltungen.jsp?personId=64858304">ekVV</a> provides an overiew of my courses (German & English) at Bielefeld University.
In addition, I am teaching through supervison of iGEM <a href="https://www.igem-bielefeld.de/">Bielefeld-CeBiTec</a> teams as outlined above.
My projects about digitial innovations in higher education were kindly supported by a fellowship from the <a href="https://www.stifterverband.org/digital-lehrfellows/2017/pucker">Stifterverband</a>.
Here is a list of my courses with links to slides and data sets:
</p>

<ul>
</li>	
<li>Applied Python Programming for Life Scientists<br>
<a href="https://pub.uni-bielefeld.de/download/2920633/2920815">Paper</a>, 
<a href="https://github.com/bpucker/APPLS">github</a>
<p style="color:green"><b>Next course: March 2020</b> <a href="https://ekvv.uni-bielefeld.de/kvv_publ/publ/vd?id=186022250">ekVV</a> given by <a href="https://www.cebitec.uni-bielefeld.de/~hschilbe">Hanna</a></p>
</li>
<li>Applied Genome Research<br>
<a href="https://doi.org/10.1515/jib-2019-0005">Paper</a>,
<a href="https://github.com/bpucker/AppliedGenomeResearch">github</a>
</li>
<li>Molecular Methods in Genome Research<br>
(funded by 
<a href="https://www.stifterverband.org/digital-lehrfellows/2017/pucker">'digi' fellowship</a> and 
<a href="https://www.genomforschung.uni-bielefeld.de/en/">Genetics & Genomics of Plants</a>)<br>
<a href="https://doi.org/10.1515/jib-2019-0005">Paper</a>,
<a href="https://github.com/bpucker/MolecularMethodsInGenomeResearch">github</a>
</li>
<li>Progress in Synthetic Biology<br>
<a href="https://ekvv.uni-bielefeld.de/kvv_publ/publ/vd?id=119368540">ekVV</a>
</li>
</ul>

<h4 id="iGEM">Synthetic Biology (iGEM)</h4>
<p>iGEM is the largest competition for synthetic biology. Student teams from all over the world participate and present their project at the annual Giant Jamboree.
<a href="https://www.igem-bielefeld.de/">iGEM Bielefeld-CeBiTec</a> participates in <a href="https://igem.org/Main_Page">iGEM</a> since 2010.
I was involved in 
<a href="http://2014.igem.org/Team:Bielefeld-CeBiTec">2014</a> (team member),
<a href="http://2016.igem.org/Team:Bielefeld-CeBiTec">2016</a> (supervision),
<a href="http://2017.igem.org/Team:Bielefeld-CeBiTec">2017</a> (supervision),
<a href="http://2018.igem.org/Team:Bielefeld-CeBiTec">2018</a> (supervision), and
<a href="http://2019.igem.org/Team:Bielefeld-CeBiTec">2019</a> (supervision). In addition, I severed as a Judge at the Giant Jamboree and mentored several other teams including
<a href="https://2019.igem.org/Team:Athens"> Athens 2019</a> and
<a href="https://2019.igem.org/Team:Thessaly">Thessaly 2019</a>.
<a href="https://after.igem.org/">After iGEM</a> selected me as 'iGEMer of the month' in September 2019.</p>

<br>

<br>


<h3 id="CV">CV</h3>
<h4 id="positions">Positions</h4>
<ul>
<li>2019-2020 <a href="https://www.ruhr-uni-bochum.de/mgpp/Seiten_en/index_e.html">Molecular Genetics and Physiology of Plants</a></li>
<li>2018-2019 <a href="https://www.genomforschung.uni-bielefeld.de/de/">Genetics and Genomics of Plants</a>, CeBiTec & Faculty of Biology (Bielefeld University)</li>
<li>2018 <a href="https://www.plantsci.cam.ac.uk/research/sambrockington">Evolution and Diversity</a>, 
<a href="https://www.plantsci.cam.ac.uk/">Department of Plant Sciences</a>, 
<a href="https://www.cam.ac.uk/">University of Cambridge</a> (UK)</a>
 funded by <a href="https://www.daad.de/en/">DAAD</a></li>
<li>2015-2017 <a href="https://www.genomforschung.uni-bielefeld.de/de/">Genome Research</a>, CeBiTec & Faculty of Biology (Bielefeld University)</li>
<li>2015 <a href="http://www.alacris.de/de/">ALACRiS Theranostics GmbH</a></li>
<li>2011-2015 <a href="https://www.uni-bielefeld.de/biologie/">Faculty of Biology</a> (Bielefeld University)</li>
</ul>

<h4 id="education">Education</h4>
<ul>
<li>05/2019: Dr. rer. nat. (very good), Faculty of Biology (Bielefeld University), thesis:
<i>De novo</i> Nd-1 genome assembly reveals genomic diversity of <i>Arabidopsis thaliana</i> and facilitates genome-wide non-canonical splice site analyses across plant species<br>
<a href="http://doi.org/10.4119/unibi/2935794">Dissertation Boas Pucker</a>
</li>
<li>2017-2019 Computational Methods for the Analysis of the Diversity and Dynamics of Genomes (DiDy) | <a href="https://didy.uni-bielefeld.de/">GRK1906</a></li>
<li>2017-2020 PhD Program <a href="http://www.uni-bielefeld.de/technische-fakultaet/promotion/promotionsstudiengaenge/Bioinformatik/">Bioinformatics</a> (Bielefeld University)</li>
<li>2016-2019 PhD Program <a href="https://www.uni-bielefeld.de/biologie/phd/index.html">Biology</a> (Bielefeld University)</li>
<li>2013-2015 Genome-Based Systems Biology <a href="https://ekvv.uni-bielefeld.de/sinfo/publ/variante/62556648?lang=en">(GBSB)</a>(Bielefeld University)</li>
<li>2010-2013 <a href="https://ekvv.uni-bielefeld.de/sinfo/publ/variante/62556648?lang=en">Biology</a> (Genetics, Cell Biology, Physiology) (Bielefeld University)</li>
<li>2009-2010 <a href="https://www.uni-duesseldorf.de/home/nc/en/studying-and-teaching-at-hhu/studying/all-courses-of-study-at-heinrich-heine-university/studiengang-informationen/studiengaenge/biochemie.html">Biochemistry</a> (Heinrich Heine University Duesseldorf)</li>
</ul>


<br>

<h4 id="Editor">Editor Activity</h4>
<p>
I am a member of the Editorial Board of 
<a href="https://bmcgenomics.biomedcentral.com/about/editorial-board">BMC Genomics</a>
 (<i>Plant Genomics</i> section).
</p>

<p>
Currently, I am a guest editor for a special issue about 
<a href="https://www.mdpi.com/journal/plants/special_issues/Harnessing_Power_Genome">Harnessing the Power of Genome and Transcriptome Assemblies</a>
 in Plants. This Special Issue is open to all research articles providing new insights into the biology of plants based on genome or transcriptome assemblies.
</p>


<br>

<h4 id="reviewer">Reviewer Activities</h4>
<p>A comprehensive list of my reviewer activities is available via <a href="https://publons.com/researcher/1008011/boas-pucker/">Publons</a>. Here is a list of journals and communities which I supported with reviews:</p>
<a href="https://www.mdpi.com/journal/agriculture">Agriculture</a>,
<a href="https://www.mdpi.com/journal/agronomy">Agronomy</a>,
<a href="https://www.mdpi.com/journal/biology">Biology</a>,
<a href="https://bmcgenomics.biomedcentral.com/">BMC Genomics</a>,
<a href="https://www.frontiersin.org/journals/plant-science#">Frontiers in Plant Sciences</a>,
<a href="https://www.mdpi.com/journal/microorganisms">Microorganisms</a>,
<a href="https://www.springer.com/journal/11033">Molecular Biology Reports</a>,
<a href="https://www.nature.com/srep/">Scientific Reports</a>,
<a href="https://www.mdpi.com/journal/genes">Genes</a>,
<a href="">Journal of Limnology and Freshwater Fisheries Research</a>,
<a href="https://peerj.com/">PeerJ</a>,
<a href="https://www.springer.com/journal/299">Plant Cell Reports</a>,
<a href="https://journals.plos.org/plosone/">PLOS ONE</a>,
<a href="https://www.springer.com">Springer Nature</a>,
<a href="https://www.mdpi.com/journal/plants">Plants</a>,
<a href="https://www.nature.com/commsbio/">Communications Biology</a>,
<a href="https://www.mdpi.com/journal/ijms">International Journal of Molecular Sciences</a>
<a href="http://www.jipb.net">Journal of Integrative Plant Biology (JIPB)</a>,
<a href="https://www.springer.com/journal/122">Theoretical and Applied Genetics (TAAG)</a>,
<a href="https://www.pyopensci.org/">pyOpenSci</a>,
<a href="https://www.mdpi.com/journal/life">life</a>,
<a href="https://www.mdpi.com/journal/biomedicines">biomedicines</a>,
<a href="https://besjournals.onlinelibrary.wiley.com/journal/2041210X">Methods in Ecology and Evolution</a>



<br><br>
<h4 id="memberships">Memberships</h4>
<ul>
<li><a href="https://community.plantae.org/user/bpucker">Plantae</a> / American Society of Plant Biologists (ASPB)</li>
<li>Deutscher Hochschul Verband (<a href="https://www.hochschulverband.de">DHV</a>)</li>
<li>Deutsche Botanische Gesellschaft (<a href="https://www.deutsche-botanische-gesellschaft.de/">DBG</a>)</li>
<li>European Society of Evolutionary Biology (<a href="https://eseb.org/">ESEB</a>)</li>
</ul>

