[Publications](https://github.com/bpucker/Boas_Pucker/edit/master/README.md#publications) - [Research Interests](https://github.com/bpucker/Boas_Pucker/edit/master/README.md#research-interests) - [Funding](https://github.com/bpucker/Boas_Pucker/edit/master/README.md#awards-funding-and-scholarships) - [CV](https://github.com/bpucker/Boas_Pucker/edit/master/README.md#cv) - [Editor/Reviewer](https://github.com/bpucker/Boas_Pucker/edit/master/README.md#editorreviewer-activity) - [Teaching](https://github.com/bpucker/Boas_Pucker/edit/master/README.md#teaching) - [Science Communication](https://github.com/bpucker/Boas_Pucker/edit/master/README.md#science-communication)


# Prof. Dr. Boas Pucker
<img src="https://www.cebitec.uni-bielefeld.de/~bpucker/BP.jpg" alt="foto" width="20%" height="20%" align="right">

+ My group: [Plant Biotechnology and Bioinformatics](https://www.tu-braunschweig.de/en/ifp/pbb) (Institute of Plant Biology & BRICS, TU Braunschweig, Germany)
+ email: b.pucker{a}tu-braunschweig.de, bpucker{a}cebitec{dot}uni-bielefeld.de
+ Google Scholar: [Boas Pucker](https://scholar.google.de/citations?user=GnnyChAAAAAJ&hl=de&oi=ao)
+ WebOfScience: [Boas Pucker](https://publons.com/researcher/1008011/boas-pucker/)
+ ORCID: [0000-0002-3321-7471](https://orcid.org/0000-0002-3321-7471)
+ Twitter: [@boas_pucker](https://twitter.com/boas_pucker)
+ Github: [bpucker](https://github.com/bpucker)
+ LinkedIn: [Boas Pucker](https://www.linkedin.com/in/boas-pucker-80aaa411b/)
+ Mastodon: [Boas Pucker](https://mastodon.social/@boas_pucker)
+ BlueSky: [Boas Pucker](https://bsky.app/profile/bpucker.bsky.social)

<b>Collaboration</b>: I am interested in collaborations involving genomics or bioinformatic analyses and matching the research interests of my group. Please send me an email.

## Research Interests

### Specialized Metabolism: flavonoid biosynthesis
Plants produce a plethora of specialized metabolites to cope with numerous environmental challenges. Examples are terpenoids, flavonoids, and betalains. We are particularly interested in flavonoids which are responsible for the pigmentation of many blue or red flowers. Flavonoids can be separated into several subgroups including flavonols, anthocyanins, proanthocyanidins, and flavones. There are multiple reasons why working on the flavonoid biosynthesis is promising.

1) The core of the flavonoid biosynthesis is a well understood model system for the regulation of biosynthesis pathways in plants. Decades of work revealed insights into the different enzymes, but also into the transcriptional regulation of the involved genes. Even Nobel Prizes have been won in association with the flavonoid biosynthesis.

2) Flavonoids have enormous potential in biotechnological applications as natural colorants and due to their high nutritional value. While the general biosynthesis pathway is well conserved across distantly related plant species, their are species-specific differences in the modification of flavonoids. Detailed knowledge about the flavonoid biosynthesis in (orphan) crop species can pave the way to a healthy nutrition. Genome editing or breeding methods can be used to improve crops with respect to their nutritional value.

3) Visible phenotypes resulting from the knock-out or increased activation of flavonoid biosynthesis genes can be helpful in the identification of promising mutants. Visible phenotypes were one reason why the flavonoid biosynthesis was established as a model system. However, coloration is not only important for basic research. Plants with altered expression of certain flavonoid biosynthesis genes show often fascinating colorations and are favored in horticultural plant species.

4) Our work investigates the Caryophyllales which represent an outstanding system to study the flavonoid biosynthesis. This order of plants is characterized by a complex pigment evolution. Briefly, specific families in the Caryophyllales show a replacement of anthocyanins (a group of flavonoids) by betalains. Betalains do not occur in plants outside the Caryophyllales and it appears that anthocyanins and betalains are mutually exclusive. The Caryophyllales are a unique opportunity to investigate the interplay of different branches of the flavonoid biosynthesis. The comparison between anthocyanin-pigmented species and species without anthocyanins reveals new insights into the biosynthesis and regulation.

Generally, we are interested in the discovery of promising biosynthesis pathways for biotechnological applications. This is not restricted to the flavonoid biosynthesis, but instead using this model system to develop new genome mining approaches. Different methods for the identification biosynthetic pathways are combined including screens and comparisons of plant genome sequences.

Selected publications:
+ Loss of anthocyanin pigmentation in the Caryophyllales: [Pucker et al., 2024](https://doi.org/10.1111/nph.19341)
+ Competition of DFR and FLS: [Choudhary & Pucker, 2023](https://doi.org/10.1101/2023.11.05.565693)
+ Loss of anthocyanins in plants: [Recinos & Pucker, 2023](https://doi.org/10.1101/2023.06.05.543820)


### Plant Genomics: Long Read Sequencing
Plant genome sequences contain the blue print for all proteins (enzymes). Sequencing and investigating genomes is an effective approach to reveal the biochemical potential of plants. Especially the correlation of genomic data (DNA) with transcriptomic (RNA) and metabolomic (chemical compounds) data sets allows the identification of biosynthesis pathways. Rapid developments of long read sequencing technologies allow the cost-effective analysis of large plant genomes. Sequencers distributed by Oxford Nanopore Technologies (ONT) are portable and can even be operated in the field. This so called nanopore sequencing approach analysis individual DNA strands. We use this technology to resolve the genome sequences of of important plant species. This is also a great opportunity for students to contribute to a genome sequencing project. Have look about our review article about the [future of long read sequencing in plant genomics](https://doi.org/10.1017/qpb.2021.18).
These genome sequencing projects have been conducted in my group or are ongoing:
+ _Ceratophyllum submersum_ (soft hornwort) is an aquatic plant that is very common in (eutrophic) ponds in Germany. We analyzed the plastome (subgenome of the chloroplast) of this species to determine its relationship to other plant species ([Meckoni et al., 2023](https://doi.org/10.1186/s13104-023-06459-z)).
+ _Digitalis purpurea_ (floxglove) is known as a medicinal plant, but also appears frequently as horticultural plant in gardens due to its colorfull and large flowers. We are interested in the flavonoid biosynthesis of this plant species with a particular focus on anthocyanins which are responsible for the redish pigmentation and also the name of the species. Read about the details in our Preprint on bioRxiv ([Wolff, Friedhoff, Horz et al., 2024](https://doi.org/10.1101/2024.02.14.580303)).
+ _Urtica dioica_ (stinging nettle) was the medicinal plant of the year in 2022. It is well known for a high flavonoid content and often used to produce tea.
+ _Aquilegia vulgaris_ (common columbine) is a horticultural plant that can be seen in many gardens showing a variety of flower colors. We are interested to understand the molecular basis of different flower colors.
+ _Victoria cruziana_ (giant water lily) is the biggest attraction in the Botanical Garden in Braunschweig. The plant is famous for huge floating leaves that can carry the weight of a small child. It produces beautiful flowers that open at night. We are interested to understand the flavonoid biosynthesis in this aquatic plant species.
+ _Rubus armeniacus_ (blackberry) is a fruit producing plant species that is common throughout Germany. It is well known for the black and sweet fruits that it produces. We are interested to study the formation of anthocyanins, a subgroup of the flavonoids, in blackberry.


### Applied Bioinformatics
Specific biological questions require the development of dedicated tools. My group writes such tools mostly in Python and R. The developed tools are freely available on GitHub ([PBB-tools](https://github.com/bpucker/PBBtools)) and on our web server ([pbb-tools](https://pbb-tools.de/)). The following tools are examples of active developments:
+ KIPEs (A): Knowledge-based Identification of Pathway Enzymes allows the automatic annotation of the proteins involved in the core steps of the flavonoid biosynthesis. This supports the identification of molecular mechanisms underlying color differences between cultivars of the same species. In addition, rapid annotation of flavonoid biosynthesis genes in novel/uncharacterized species becomes convenient. Read the [KIPEs paper](https://www.mdpi.com/2223-7747/9/9/1103/htm), the [KIPEs3 paper](https://doi.org/10.1371/journal.pone.0294342) or download KIPEs from [GitHub](https://github.com/bpucker/KIPEs).
+ MGSE (B): Mapping-based Genome Size Estimation is a novel approach to infer the true genome size of a species based on sequence reads. This approach harnesses the equal representation of all regions (even repeats) in the read set. The average number of sequence reads (coverage) is estimated based on single copy regions in a reference genome sequence. Dividing the combined coverage of all positions in an assembly by this sequencing depth results in the genome size estimation. Read the [MGSE preprint](https://doi.org/10.1101/607390) or download MGSE from [GitHub](https://github.com/bpucker/MGSE).
+ NAVIP (C): Neighborhood-Aware Variant Impact Predictor enables the prediction of functional consequences arising from sequence variants between a sequenced sample and a reference. In contrast to many established tools, NAVIP considers all variants in one gene at the same time when predicting the potential effect of sequence variants. Read the [NAVIP preprint](https://doi.org/10.1101/596718 ) or download NAVIP from [GitHub](https://github.com/bpucker/NAVIP).
+ MYB_annotator (D): This tool enables the automatic identification and annotation of MYBs in a novel transcriptome/genome sequence assembly of a plant species. The identified candidates are functionally annotated based on orthology to previously characterized sequences. Read the [MYB_annotator paper](https://doi.org/10.1186/s12864-022-08452-5) or download the tool from [GitHub](https://github.com/bpucker/MYB_annotator).
+ bHLH_annotator (E): This tool enables the automatic identification and annotation of bHLHs in a novel transcriptome/genome sequence assembly of a plant species. The identified candidates are functionally annotated based on orthology to previously characterized sequences. Read the [bHLH_annotator paper](https://doi.org/10.1186/s12864-023-09877-2) or download the tool from [GitHub](https://github.com/bpucker/bHLH_annotator).


Studied Plant Species: <i>Arabidopsis thaliana</i>, <i>Arabidopsis halleri</i>, <i>Beta vulgaris</i>, <i>Beta coroliflora</i>, <i>Beta lomatogona</i>, <i>Brassica napus</i>, <i>Cicer arietinum</i>, <i>Croton tiglium</i>, <i>Digitalis purpurea</i>, <i>Dioscorea dumetorum</i>, <i>Kewa caespitosa</i>, <i>Macarthuria australis</i>, <i>Medicago truncatula</i>, <i>Musa acuminata</i>, <i>Pharnaceum exiguum</i>, <i>Simmondsia chinensis</i>, <i>Urtica dioica</i>, <i>Vitis vinifera</i> various other plant species especially members of the Caryophyllales.


## Awards, funding, and scholarships
+ 2024 Financial Support for SynBio2024
+ 2023 DFG Research Project ['Phylogenomic elucidation of withanolide biosynthesis in Nightshade plants'](https://gepris.dfg.de/gepris/projekt/516566273)
+ 2023 Financial support for iGEM2023
+ 2022 Freiraum2022 Fellowship for ['Data Literacy in Genome Research'](https://www.tu-braunschweig.de/en/ifp/pbb/teaching/dalip)
+ 2022 Financial support for iGEM2022
+ 2019 DFG Research Fellowship for ['Understanding the convergent evolution of a complex metabolic trait, the Betalains, through comparative genomics and co-expression networks'](https://gepris.dfg.de/gepris/projekt/436841671) in Evolution and Diversity, Department of Plant Sciences, University of Cambridge, UK
+ 2019 Research Fellowship for Plant Genomics and Bioinformatics
+ 2018 Fellowship for Digital Innovations in Academic Teaching([digital laboratory notebook](https://www.stifterverband.org/digital-lehrfellows/2017/pucker))
+ 2018 DAAD scholarship (University of Cambridge, UK)
+ 2015 Award for excellent master's thesis ([Friedrich Wilhelm Helweg Foundation](https://www.im.nrw/friedrich-wilhelm-helweg-stiftung))
+ 2014 [Deutschlandstipendium](https://www.deutschlandstipendium.de/de/english-1700.html)


## Teaching
### Teaching activities at TU Braunschweig
Please see my website about courses at TU Braunschweig for latest updates: [BP teaching at TUBS](https://www.tu-braunschweig.de/en/ifp/pbb/teaching).


### Teaching activities at Bielefeld University
The [ekVV](https://ekvv.uni-bielefeld.de/kvv_publ/publ/Lehrende_Veranstaltungen.jsp?personId=64858304) provides an overiew of my courses (German & English) at Bielefeld University. In addition, I am teaching through supervison of iGEM <a href="https://www.igem-bielefeld.de/">Bielefeld-CeBiTec</a> teams as outlined above. My projects about digitial innovations in higher education were kindly supported by a fellowship from the [Stifterverband](https://www.stifterverband.org/digital-lehrfellows/2017/pucker).
Here is a list of my courses with links to slides and data sets:

+ Applied Python Programming for Life Scientists ([Paper](https://pub.uni-bielefeld.de/download/2920633/2920815), [GitHub](https://github.com/bpucker/APPLS))
+ Applied Genome Research ([Paper](https://doi.org/10.1515/jib-2019-0005), [GitHub](https://github.com/bpucker/AppliedGenomeResearch))
+ Molecular Methods in Genome Research (funded by ['digi' fellowship](https://www.stifterverband.org/digital-lehrfellows/2017/pucker) and Genetics & Genomics of Plants, [Paper](https://doi.org/10.1515/jib-2019-0005), [GitHub](https://github.com/bpucker/MolecularMethodsInGenomeResearch))
+ Progress in Synthetic Biology ([ekVV](https://ekvv.uni-bielefeld.de/kvv_publ/publ/vd?id=119368540))


### Synthetic Biology (iGEM)
iGEM is the largest competition for synthetic biology. Student teams from all over the world participate and present their synthetic biology projects at the annual Giant Jamboree. I participated in the teams [iGEM Bielefeld-CeBiTec](https://www.igem-bielefeld.de/) and [iGEM TU Braunschweig](https://www.tu-braunschweig.de/en/ifp/pbb/synbio) in multiple years:
+ [2014](http://2014.igem.org/Team:Bielefeld-CeBiTec) (team member)
+ [2016](http://2016.igem.org/Team:Bielefeld-CeBiTec) (supervision)
+ [2017](http://2017.igem.org/Team:Bielefeld-CeBiTec) (supervision)
+ [2018](http://2018.igem.org/Team:Bielefeld-CeBiTec) (supervision)
+ [2019](http://2019.igem.org/Team:Bielefeld-CeBiTec) (supervision); mentoring: [Athens](https://2019.igem.org/Team:Athens) and [Thessaly](https://2019.igem.org/Team:Thessaly); After iGEM selected me as 'iGEMer of the month' in September 2019
+ 2020 (judge)
+ 2021 (judge); mentoring [RUBochum](https://2021.igem.org/Team:RUBochum)
+ [2022](https://2022.igem.wiki/tu-braunschweig/) (PI)
+ [2023](https://2023.igem.wiki/tu-braunschweig/) (PI)


## CV
### Positions
+ since 2022 Head of Technology Transfer Center 'Plant Genomics and Applied Bioinformatics'
+ since 2021 Professor at TU Braunschweig, [Plant Biotechnology and Bioinformatics](https://www.tu-braunschweig.de/en/ifp/pbb), Institute of Plant Biology & BRICS
+ 2020-2021 [Evolution and Diversity](http://brockingtonlab.co.uk/team.html), Department of Plant Sciences, University of Cambridge (UK) funded by DFG
+ 2019-2020 [Molecular Genetics and Physiology of Plants](https://www.ruhr-uni-bochum.de/mgpp/Seiten_en/index_e.html)
+ 2018-2019 [Genetics and Genomics of Plants](https://www.uni-bielefeld.de/fakultaeten/biologie/forschung/arbeitsgruppen/ggp/), CeBiTec & Faculty of Biology (Bielefeld University)
+ 2018 [Evolution and Diversity](http://brockingtonlab.co.uk/team.html), Department of Plant Sciences, University of Cambridge (UK) funded by DAAD
+ 2015-2017 [Genome Research](https://www.genomforschung.uni-bielefeld.de/de/), CeBiTec & Faculty of Biology (Bielefeld University)
+ 2015 [ALACRiS Theranostics GmbH](http://www.alacris.de/de)
+ 2011-2015 [Faculty of Biology](https://www.uni-bielefeld.de/biologie/) (Bielefeld University)

### Education
+ 05/2019: Dr. rer. nat. (very good), Faculty of Biology (Bielefeld University), thesis:<i>De novo</i> Nd-1 genome assembly reveals genomic diversity of <i>Arabidopsis thaliana</i> and facilitates genome-wide non-canonical splice site analyses across plant species ([Dissertation Boas Pucker](http://doi.org/10.4119/unibi/2935794))
+ 2017-2019 Computational Methods for the Analysis of the Diversity and Dynamics of Genomes (DiDy) | GRK1906
+ 2017-2020 PhD Program 'Bioinformatics' (Bielefeld University)
+ 2016-2019 PhD Program 'Biology' (Bielefeld University)
+ 2013-2015 Genome-Based Systems Biology (GBSB) (Bielefeld University)
+ 2010-2013 Biology (Genetics, Cell Biology, Physiology) (Bielefeld University)
+ 2009-2010 Biochemistry (Heinrich Heine University Duesseldorf)


## Editor/Reviewer Activity
+ Member of the [Editorial Board](https://bmcgenomics.biomedcentral.com/about/editorial-board) of BMC Genomics (<i>Plant Genomics</i> section).
+ Associate Editor at Frontiers ([loop profile](https://loop.frontiersin.org/people/456636/overview))
+ Reviewer Activities: A detailed list of my reviewer activities is available via [WebOfScience](https://publons.com/researcher/1008011/boas-pucker/) or [ORCID](https://orcid.org/0000-0002-3321-7471). Here is a list of selected journals and communities which I supported with reviews:
Nature Catalysis, Nature Communications, The Plant Journal, Genomics, Bioinformatics, GigaScience, Horticultural Research, Journal of Experimental Botany, Molecular Plant, BMC Plant Biology, BMC Genomics, Frontiers in Plant Sciences, Molecular Biology Reports, Scientific Reports, PeerJ, Plant Cell Reports, PLOS ONE, Communications Biology, Journal of Integrative Plant Biology (JIPB), Theoretical and Applied Genetics (TAAG), pyOpenSci, Methods in Ecology and Evolution


### Memberships
+ 06/2023-present Verband Biologie, Biowiss. und Biomed in Deutschland e.V (VBIO)
+ 05/2023-present DSI Scientific Network
+ 01/2023-present German Association for Synthetic Biology (GASB)
+ 07/2020-present European Society of Evolutionary Biology (ESEB)
+ 02/2020-present [Plantae](https://community.plantae.org/user/bpucker) / American Society of Plant Biologists (ASPB)
+ 12/2019-present German Society for Plant Sciences (DBG)
+ 2019-present bts e.V. (biotechnology students' initiative)
+ Freunde des Braunschweiger Botanischen Gartens e.V.
+ Braunschweiger Hochschulbund e.V. (BHB)
+ Deutscher Hochschul Verband (DHV)

## Science Communication
+ [FlavonoidFriday](https://github.com/bpucker/FlavonoidFriday): Novel findings about the flavonoid biosynthesis are shared via social media on a weekly basis. Details about these posts are available in the corresponding GitHub repository.
+ [MybMonday](https://github.com/bpucker/MybMonday): Interesting facts about the R2R3-MYB transcription factor family is shared via social media on a weekly basis. Details about these posts are available in the corresponding GitHub repository.



## Publications
A full list of publications is available at my [Google scholar profile](https://scholar.google.com/citations?user=1CbAO2UAAAAJ&hl=en).
+ x
+ 
+ 
+ 
+ 
+

**2022**
+ Movahedi A, Wei H, Pucker B, Sun W, Li W, Yang L, Zhuge Q (2022). Regulation of poplar isoprenoid biosynthesis by methylerythritol phosphate and mevalonic acid pathways interactions. Front. Plant Sci. 13:968780. doi: [10.3389/fpls.2022.968780](https://doi.org/10.3389/fpls.2022.968780).
+ Rajput, R., Tyagi, S., Naik, J. Pucker, B., Stracke, R., Pandey, A. The R2R3-MYB gene family in Cicer arietinum: genome-wide identification and expression analysis leads to functional characterization of proanthocyanidin biosynthesis regulators in the seed coat. Planta 256, 67 (2022). doi: [10.1007/s00425-022-03979-z](https://doi.org/10.1007/s00425-022-03979-z).
+ Mengist MF, Grace MH, Mackey T, Munoz B, Pucker B, Bassil N, Luby C, Ferruzzi M, Lila MA and Iorizzo M (2022) Dissecting the genetic basis of bioactive metabolites and fruit quality traits in blueberries (Vaccinium corymbosum L.). Front. Plant Sci. 13:964656. doi: [10.3389/fpls.2022.964656](https://doi.org/10.3389/fpls.2022.964656).
+ Schilbert H. M.*, Pucker B.*, Ries D., Viehöver P., Micic Z., Dreyer F., Beckmann K., Wittkop B., Weisshaar B., Holtgräwe D.. Mapping-by-sequencing reveals genomic regions associated with seed quality parameters in Brassica napus. Genes 2022, 13(7), 1131; doi: [10.3390/genes13071131](https://doi.org/10.3390/genes13071131).
+ Ballmann, R.; Hotop, S.; Bertoglio, F.; Steinke, S.; Heine, P.A.; Chaudhry, M.; Jahn, D.; Pucker, B.; Baldanti, F.; Piralla, A.; Schubert, M.; Čičin-Šain, L.; Hust, M.; Dübel, S. ORFeome Phage Display Reveals a Major Immunogenic Epitope on the S2 Subdomain of SARS-Cov-2 Spike Protein. Viruses 2022, 14, 1326. doi: [10.3390/v14061326](https://doi.org/10.3390/v14061326).
+ Pucker B (2022). Automatic identification and annotation of MYB gene family members in plants. BMC Genomics 23, 220 (2022). doi: [10.1186/s12864-022-08452-5](https://doi.org/10.1186/s12864-022-08452-5).
+ Pucker B, Irisarri I, de Vries J and Xu B (2022). Plant genome sequence assembly in the era of long reads: Progress, challenges and future directions. Quantitative Plant Biology, 3, E5. doi: [10.1017/qpb.2021.18](https://www.doi.org/10.1017/qpb.2021.18).
+ Pucker B & Selmar D (2022). Biochemistry and Molecular Basis of Intracellular Flavonoid Transport in Plants. Plants 2022, 11, 963. doi: [10.3390/plants11070963](https://doi.org/10.3390/plants11070963).
+ Sielemann K, Pucker B, Schmidt N, Viehoever P, Weisshaar B, Heitkam T, Holtgraewe D (2021). Complete pan-plastome sequences enable high resolution phylogenetic classification of sugar beet and closely related crop wild relatives. BMC Genomics 23, 113 (2022). doi: [10.1186/s12864-022-08336-8](https://doi.org/10.1186/s12864-022-08336-8).


<ul>

<li>
<b>Pucker B</b> & Brockington S F (2022). The evidence for anthocyanins in the betalain-pigmented genus Hylocereus is weak. 
BMC Genomics 23, 739. doi: https://doi.org/10.1186/s12864-022-08947-1. <br>
<a href="https://doi.org/10.1186/s12864-022-08947-1">full text</a>
<a href="https://bmcgenomics.biomedcentral.com/counter/pdf/10.1186/s12864-022-08947-1.pdf">PDF</a>
</li>

<li>
<b>Pucker B</b> (2021). Automatic identification and annotation of MYB gene family members in plants. 
bioRxiv 2021.10.16.464636; doi: https://doi.org/10.1101/2021.10.16.464636 <br>
<a href="https://doi.org/10.1101/2021.10.16.464636">full text</a>
<a href="https://www.biorxiv.org/content/10.1101/2021.10.16.464636v1.full.pdf">PDF</a>
</li>

<li>
Sielemann K, <b>Pucker B</b>, Schmidt N, Viehoever P, Weisshaar B, Heitkam T, Holtgraewe D (2021). 
Complete pan-plastome sequences enable high resolution phylogenetic classification of sugar beet and closely related crop wild relatives. 
bioRxiv 2021.10.08.463637; doi: 10.1101/2021.10.08.463637. <br>
<a href="https://doi.org/10.1101/2021.10.08.463637">full text</a>
<a href="https://www.biorxiv.org/content/10.1101/2021.10.08.463637v1.full.pdf">PDF</a>
</li>


<li>
Movahedi A, Wei H, <b>Pucker B</b>, Sun W, Li W, Yang L, Zhuge Q (2021). 
Regulation of poplar isoprenoid biosynthesis by methylerythritol phosphate and mevalonic acid pathways interactions. 
bioRxiv 2020.07.22.216804; doi: 10.1101/2020.07.22.216804. <br>
<a href="https://doi.org/10.1101/2020.07.22.216804">full text</a>
<a href="https://www.biorxiv.org/content/10.1101/2020.07.22.216804v3.full.pdf">PDF</a>
</li>


<li>Chibani K., <b>Pucker B.</b>, Dietz K.-J., Cavanagh A. (2021). Genome-wide analysis and transcriptional regulation of the typical and atypical thioredoxins in <i>Arabidopsis thaliana</i>. doi:10.1002/1873-3468.14197. <br>
<a href="https://doi.org/10.1002/1873-3468.14197">full text</a>
<a href="https://febs.onlinelibrary.wiley.com/doi/epdf/10.1002/1873-3468.14197">PDF</a>
</li>


<li>Schilbert H, Kleinb&ouml;lting N, Weisshaar B, <b>Pucker B.</b> (2021). <i>Arabidopsis thaliana</i> methylation pattern analysis based on ONT sequence reads. Bielefeld University. <br>
<a href="https://pub.uni-bielefeld.de/record/2956654">full text</a>
<a href="https://pub.uni-bielefeld.de/record/2956654">PUB</a>
</li>


<li><b>Pucker, B.</b>*, Kleinb&ouml;lting, N.* and Weisshaar, B.  (2021).  
Large scale genomic rearrangements in selected Arabidopsis thaliana T-DNA lines are caused by T-DNA insertion mutagenesis. BMC Genomics. doi: 10.1186/s12864-021-07877-8<br>
<a href="https://doi.org/10.1186/s12864-021-07877-8">full text</a>
<a href="https://bmcgenomics.biomedcentral.com/track/pdf/10.1186/s12864-021-07877-8.pdf">PDF</a>
<a href="https://www.biorxiv.org/content/10.1101/2021.03.03.433755v3">preprint</a>
<a href="https://pub.uni-bielefeld.de/record/2952434">PUB</a>
<a href="https://github.com/bpucker/GKseq">github</a>
<a href="https://github.com/nkleinbo/loreta">github</a>
<br>
*shared first authorship
</li>


<li><b>Pucker B.</b>, Singh H.B., Kumari M., Khan M.I., Brockington S.F. (2021)
The report of anthocyanins in the betalain-pigmented genus <i>Hylocereus</i> is not well evidenced and is not a strong basis to refute the mutual exclusion paradigm.
BMC Plant Biology 21(1): 297. <br>
<a href="https://doi.org/10.1186/s12870-021-03080-9">full text</a>
<a href="https://bmcplantbiol.biomedcentral.com/track/pdf/10.1186/s12870-021-03080-9.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2955954">PUB</a>
<a href="https://pub.uni-bielefeld.de/record/2946374">data</a>
<br>
</li>


<li>Montero, H., Lee, T., <b>Pucker, B.</b>, Ferreras-Garrucho, G., Oldroyd, G., Brockington, S.F., Miyao, A. and Paszkowski, U.  
A mycorrhiza-associated receptor-like kinase with an ancient origin in the green lineage. 
PNAS (2021). doi:10.1073/pnas.2105281118<br>
<a href="https://www.pnas.org/content/118/25/e2105281118">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://www.pnas.org/content/pnas/118/25/e2105281118.full.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2955558">PUB</a>
</li>


<li>Naik, J., Rajput, R., <b>Pucker, B.</b>, Stracke, R. and Pandey, A. 
The R2R3-MYB transcription factor MtMYB134 orchestrates flavonol biosynthesis in Medicago truncatula. 
Plant Mol Biol (2021). doi:10.1007/s11103-021-01135-x<br>
<a href="https://doi.org/10.1007/s11103-021-01135-x">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://link.springer.com/content/pdf/10.1007/s11103-021-01135-x.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2952659">PUB</a>
</li>



<li><b>Pucker, B.</b>*, Kleinb&ouml;lting, N.* and Weisshaar, B.  (2021).  
Large scale genomic rearrangements in selected Arabidopsis thaliana T-DNA lines are caused by T-DNA insertion mutagenesis. doi: 10.1101/2021.03.03.433755<br>
<a href="https://www.biorxiv.org/content/10.1101/2021.03.03.433755v2">full text</a>
<a href="https://www.biorxiv.org/content/10.1101/2021.03.03.433755v2.full.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2952434">PUB</a>
<a href="https://github.com/bpucker/GKseq">github</a>
<a href="https://github.com/nkleinbo/loreta">github</a>
<br>
*shared first authorship
</li>


<li>Sielemann, K., Weisshaar, B. and <b>Pucker, B.</b> (2021).  
Reference-based QUantification Of gene Dispensability (QUOD). Plant Methods 17, 18. doi: 10.1186/s13007-021-00718-5<br>
<a href="https://doi.org/10.1186/s13007-021-00718-5">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://doi.org/10.1101/2020.04.28.065714">bioRxiv</a>
<a href="https://plantmethods.biomedcentral.com/track/pdf/10.1186/s13007-021-00718-5.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2943037">PUB</a>
<a href="https://github.com/KatharinaFrey/QUOD">github</a>
</li>

<li>
Xu B., Taylor L., <b>Pucker B.</b>, Feng T., Glover B. J. and Brockington S. (2021).
The land plant-specific MIXTA-MYB lineage is implicated in the early evolution of the plant cuticle and the colonization of land. New Phytologist. doi:10.1111/nph.16997.<br>
<a href="https://doi.org/10.1111/nph.16997">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://pub.uni-bielefeld.de/record/2946565">PUB</a>
</li>


<li>
<b>Pucker B.</b>, Schwandner A., Becker S., Hausmann L., Vieh&ouml;ver P., T&ouml;pfer R., Weisshaar B., Holtgr&auml;we D. (2020).
RNA-Seq Time Series of Vitis vinifera Bud Development Reveals Correlation of Expression Patterns with the Local Temperature Profile.
 Plants 2020, 9(11), 1548; doi:10.3390/plants9111548.<br>
 <a href="https://www.mdpi.com/2223-7747/9/11/1548/htm">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://doi.org/10.1101/2020.10.18.344176">preprint</a>
<a href="https://www.mdpi.com/2223-7747/9/11/1548/pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2946639">PUB</a>
</li>


<li>
<b>Pucker B.</b> and Brockington S. (2020). Pitaya transcriptome assemblies and investigation of transcript abundances. Bielefeld University. doi: 10.4119/unibi/2946374.<br>
<a href="https://pub.uni-bielefeld.de/record/2946374">PUB</a>
<a href="https://github.com/bpucker/pitaya">github</a>
</li>


<li><b>Pucker B.</b>, Pandey A., Weisshaar B. and Stracke R. (2020).  
The <i>R2R3-MYB</i> gene family in banana (Musa acuminata): genome-wide identification, classification and expression patterns.
 PLOS ONE 15(10): e0239275. doi:10.1371/journal.pone.0239275<br>
<a href="https://doi.org/10.1371/journal.pone.0239275">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://doi.org/10.1101/2020.02.03.932046">preprint</a>
<a href="https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0239275&type=printable">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2940694">PUB</a>
<a href="https://github.com/bpucker/bananaMYB">github</a>
</li>


<li>Sielemann, K.*, Hafner, A.* and <b>Pucker B.</b> (2020).  
The Reuse of Public Datasets in the Life Sciences: Potential Risks and Rewards.  PeerJ 8:e9954. doi: 10.7717/peerj.9954<br>
<a href="https://doi.org/10.7717/peerj.9954">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://peerj.com/articles/9954.pdf">PDF</a>
<a href="https://www.preprints.org/manuscript/202002.0146/v2">preprint</a>
<a href="https://peerj.com/articles/9954.pdf">PUB</a>
*shared first authorship
</li>


<li><b>Pucker, B.</b>,  Reiher, F. and Schilbert, H. M.   
Automatic identification of players in the flavonoid biosynthesis with application on the biomedicinal plant <i>Croton tiglium</i>. 
Plants 2020, 9, 1103. doi:10.3390/plants9091103.<br>
<a href="https://www.mdpi.com/2223-7747/9/9/1103/htm">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://www.mdpi.com/2223-7747/9/9/1103/pdf">PDF</a>
<a href="https://doi.org/10.1101/2020.06.27.175067">bioRxiv</a>
<a href="https://pub.uni-bielefeld.de/record/2944597">PUB</a>
<a href="https://github.com/bpucker/KIPEs">github</a>
</li>


<li>
Pellegrinelli V., Rodriguez-Cuenca S., Rouault C., Schilbert H., Virtue S., Moreno-Navarrete J. M.,
Bidault G., del Carmen Vazquez Borrego M., Dias A. R., <b>Pucker B.</b>,
Dale M., Campbell M., Carobbio S., Aron-Wisnewsky J., Mora S., Masiero M., Emmanouilidou A.,
Mukhopadhyay S., Dougan G., den Hoed M., Loos R., Fernandez-Real J. M., Chiarugi D., Clement K. and Vidal-Puig T. 
Dysregulation of macrophage PEPD in obesity determines adipose tissue fibro-inflammation and insulin resistance.
doi:10.21203/rs.3.rs-57182.<br>
<a href="https://doi.org/10.21203/rs.3.rs-57182/v1">full text</a>
<a href="https://www.researchsquare.com/article/rs-57182/v1.pdf">PDF</a>
</li>


<li>Choi, J., Lee, T., Cho, J., Servante, E., <b>Pucker, B.</b>, Summers, W., et al.   
The negative regulator SMAX1 controls mycorrhizal symbiosis and strigolactone biosynthesis in rice. 
Nature Communications 11, 2114 (2020). doi:10.1038/s41467-020-16021-1.<br>
<a href="https://www.nature.com/articles/s41467-020-16021-1">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://www.repository.cam.ac.uk/handle/1810/304408">Apollo (free full text)</font></a>
<a href="https://pub.uni-bielefeld.de/record/2943062">PUB</a>
</li>


<li>Schilbert H. M., Rempel A. and <b>Pucker, B.</b> (2020).  
Comparison of read mapping and variant calling tools for the analysis of plant NGS data. Plants. doi:10.3390/plants9040439.<br>
<a href="https://www.mdpi.com/2223-7747/9/4/439/htm">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://doi.org/10.1101/2020.03.10.986059">bioRxiv</a>
<a href="https://www.biorxiv.org/content/10.1101/2020.03.10.986059v2.full.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2942341">PUB</a>
<a href="https://doi.org/10.4119/unibi/2942069">data</a>
</li>


<li>Siadjeu C.*, <b>Pucker B.*</b>, Viehoever P., Albach D. and Weisshaar B. (2020).  
High contiguity de novo genome sequence assembly of Trifoliate yam (Dioscorea dumetorum) using long read sequencing.
Genes. doi:10.3390/genes11030274.<br>
<a href="https://www.mdpi.com/2073-4425/11/3/274/htm">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://doi.org/10.1101/2020.01.31.928630">bioRxiv</a>
<a href="https://www.biorxiv.org/content/10.1101/2020.01.31.928630v3.full.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2940688">PUB</a>
<a href="https://github.com/bpucker/yam">github</a>
<a href="https://dx.doi.org/10.17504/protocols.io.bcvyiw7w">DNA extraction protocol</a>
<a href="https://doi.org/10.4119/unibi/2941469">data (PUB)</a>
<br>
*shared first authorship
</li>

<li>Frey K. & <b>Pucker, B.</b> (2020).  
Animal, fungi, and plant genome sequences harbour different non-canonical splice sites. Cells. doi:10.3390/cells9020458.<br>
<a href="https://www.mdpi.com/2073-4409/9/2/458">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://www.mdpi.com/2073-4409/9/2/458/pdf">PDF</a>
<a href="https://doi.org/10.1101/616565">bioRxiv</a>
<a href="https://pub.uni-bielefeld.de/record/2935311">PUB</a>
<a href="https://github.com/bpucker/ncss2018">github</a>
<a href="https://pub.uni-bielefeld.de/record/2934220">data (fungi)</a>
<a href="https://pub.uni-bielefeld.de/record/2934226">data (animals)</a>
</li>

<li>
Holtgr&auml;we, D., Rosleff S&ouml;rensen, T., Hausmann, L., <b>Pucker, B.</b>, Vieh&ouml;ver, P., T&ouml;pfer, R., Weisshaar, B. (2020).
A Partially Phase-Separated Genome Sequence Assembly of the Vitis Rootstock 'B&ouml;rner' 
(<i>Vitis riparia</i> x <i>Vitis cinerea</i>) and its Exploitation for Marker Development and Targeted Mapping. 
Front. Plant Sci. doi:10.3389/fpls.2020.00156.<br>
<a href="https://doi.org/10.3389/fpls.2020.00156">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/25/854687.full-text.pdf">PDF</a>
<a href="https://www.biorxiv.org/content/10.1101/854687v1">bioRxiv</a>
<a href="https://pub.uni-bielefeld.de/record/2939244">PUB</a>
<a href="http://doi.org/10.4119/unibi/2938185">data set 1</a>
<a href="http://doi.org/10.4119/unibi/2938180">data set 2</a>
<a href="https://doi.org/10.4119/unibi/2938178">data set 3</a>
</li>


<li> Busche M*, <b>Pucker B*</b>, Vieh&ouml;ver P, Weisshaar B, Stracke R. (2020).
Genome Sequencing of Musa acuminata Dwarf Cavendish Reveals a Duplication of a Large Segment of Chromosome 2. 
G3: Genes|Genomes|Genetics. doi:10.1534/g3.119.400847.<br>
<a href="https://doi.org/10.1534/g3.119.400847">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/691923.full-text.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2938807">PUB</a>
<a href="https://www.biorxiv.org/content/10.1101/691923v4.full">bioRxiv</a>
<a href="https://github.com/bpucker/banana">github</a>
<a href="https://pub.uni-bielefeld.de/record/2937697">genome assemblies</a>
<a href="https://pub.uni-bielefeld.de/record/2936278">sequence variants</a>
<br>
*shared first authorship
</li>


<li>
Sheehan, H., Feng, T., Walker-Hale, N., Lopez-Nieves, S., <b>Pucker, B.</b>, Guo, R., Yim, W. C., Badgami, R., Timoneda, A., Zhao, L., Tiley, H., Copetti, D., Sanderson, M. J., Cushman, J. C., Moore, M. J., Smith, S. A., Brockington, S. F. 
Evolution of L-DOPA 4,5-dioxygenase activity allows for recurrent specialisation to betalain pigmentation in Caryophyllales. 
New Phytologist. 2020. doi:10.1111/nph.16089<br>
<a href="https://doi.org/10.1111/nph.16089">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://nph.onlinelibrary.wiley.com/doi/epdf/10.1111/nph.16089">PDF</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/31369159">PubMed</a>
<a href="https://pub.uni-bielefeld.de/record/2936783">PUB</a>
<a href="https://www.repository.cam.ac.uk/handle/1810/295044">Apollo</a>
</li>


<li>
<b>Pucker B</b> & Schilbert H.
Genomics and Transcriptomics Advance in Plant Sciences. 
Molecular Approaches in Plant Biology and Environmental Challenges. 
Springer. 2019. ISBN 978-981-15-0690-1. doi:10.1007/978-981-15-0690-1<br>
<a href="https://www.springer.com/gp/book/9789811506895">springer<font color="blue">(peer-reviewed book chapter)</font></a>
<a href="https://pub.uni-bielefeld.de/record/2938013">PUB</a>
</li>

<li>
<b>Pucker B</b>, R&uuml;ckert C, Stracke R, Vieh&ouml;ver P, Kalinowski J, Weisshaar B. 
Twenty-Five Years of Propagation in Suspension Cell Culture Results in Substantial Alterations of the Arabidopsis Thaliana Genome. 
Genes. 2019. doi:10.3390/genes10090671<br>
<a href="https://www.mdpi.com/2073-4425/10/9/671">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://www.mdpi.com/2073-4425/10/9/671/pdf">PDF</a>
<a href="https://www.biorxiv.org/content/10.1101/710624v3.full">bioRxiv</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/31480756">PubMed</a>
<a href="https://pub.uni-bielefeld.de/record/2936644">PUB</a>
<a href="https://github.com/bpucker/At7">github</a>
</li>


<li>
Timoneda, A., Feng, T., Sheehan, H., Walker-Hale, N., <b>Pucker, B.</b>, Lopez-Nieves, S., Guo, R., Brockington, S. (2019).
The evolution of betalain biosynthesis in Caryophyllales.
New Phytologist. doi:10.1111/nph.15980<br>
<a href="https://doi.org/10.1111/nph.15980">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://nph.onlinelibrary.wiley.com/doi/epdf/10.1111/nph.15980">PDF</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/31172524">PubMed</a>
<a href="https://pub.uni-bielefeld.de/record/2936079">PUB</a>
</li>

<li>
 <b>Pucker, B.</b> Schilbert, H.M., Schumacher, S.F. (2019). 
Integrating Molecular Biology and Bioinformatics Education. 
Journal of Integrative Bioinformatics. doi:10.1515/jib-2019-0005.<br>
<a href="https://doi.org/10.1515/jib-2019-0005">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://www.degruyter.com/downloadpdf/j/jib.2019.16.issue-3/jib-2019-0005/jib-2019-0005.xml">PDF</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/31145692">PubMed</a>
<a href="https://www.preprints.org/manuscript/201811.0183/v2">preprint</a>
<a href="https://pub.uni-bielefeld.de/record/2935844">PUB</a><br>
<a href="https://github.com/bpucker/AppliedGenomeResearch">Applied Genome Research (github)</a>
<a href="https://github.com/bpucker/MolecularMethodsInGenomeResearch">Molecular Methods in Genome Research (github)</a>
</li>

<li><b>Pucker, B.</b>, Feng, T., Brockington, S. (2019). 
Next generation sequencing to investigate genomic diversity in Caryophyllales.
bioRxiv 646133; doi:10.1101/646133.
<br>
<a href="https://www.biorxiv.org/content/10.1101/646133v3.full">full text</a>
<a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/13/646133.full.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2935737">PUB</a>
<a href="https://docs.cebitec.uni-bielefeld.de/s/pZ4kGpPEDtTPgjW">data</a>
<a href="https://github.com/bpucker/GenomeAssemblies2018">github</a>
</li>

<li><b>Pucker, B.</b>, Holtgraewe, D., Stadermann, K. B., Frey, K., Huettel, B., Reinhardt, R., & Weisshaar, B. (2019). 
A Chromosome-level Sequence Assembly Reveals the Structure of the <i>Arabidopsis thaliana</i> Nd-1 Genome and its Gene Set. PLOS ONE: e0216233. doi: 10.1371/journal.pone.0216233<br>
<a href="https://doi.org/10.1371/journal.pone.0216233">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0216233&type=printable">PDF</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/31112551">PubMed</a>
<a href="https://www.biorxiv.org/content/10.1101/407627v3.full">bioRxiv</a>
<a href="https://pub.uni-bielefeld.de/record/2930926">PUB</a>
<a href="https://doi.org/10.5447/IPK/2019/4">data</a>
<a href="https://github.com/bpucker/Nd1_PacBio">github</a>
</li>

<li><b>Pucker, B.</b> (2019).  
Mapping-based genome size estimation. bioRxiv. doi:10.1101/607390.<br>
<a href="https://www.biorxiv.org/content/10.1101/607390v1.full">full text</a>
<a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/607390.full-text.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2934975">PUB</a>
<a href="https://github.com/bpucker/MGSE">github (MGSE)</a>
</li>

<li>Baasner, J.-S., Howard, D., <b>Pucker, B.</b> (2019).  
Influence of neighboring small sequence variants on functional impact prediction. bioRxiv. doi:10.1101/596718.<br>
<a href="https://www.biorxiv.org/content/10.1101/596718v2.full">full text</a>
<a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/27/596718.full-text.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2934778">PUB</a>
<a href="https://github.com/bpucker/NAVIP">github (NAVIP)</a>
<a href="https://github.com/bpucker/variant_calling">github (other scripts)</a>
</li>

<li><b>Pucker, B.</b> (2019). <i>De novo</i> Nd-1 genome assembly reveals genomic diversity of <i>Arabidopsis thaliana</i> and facilitates genome-wide non-canonical splice site analyses across plant species. Dissertation. doi:10.4119/unibi/2935794.<br>
<a href="https://pub.uni-bielefeld.de/download/2935794/2935982/Pucker_Dissertation.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2935794">PUB</a>
<a href="http://doi.org/10.4119/unibi/2935794">DOI</a>
</li>

<li>
 <b>Pucker, B.</b> and Brockington, S.F. (2018). 
Genome-wide analyses supported by RNA-Seq reveal non-canonical splice sites in plant genomes. 
BMC Genomics. 2018;19(1). doi:10.1186/s12864-018-5360-z.<br>
<a href="https://doi.org/10.1186/s12864-018-5360-z">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://bmcgenomics.biomedcentral.com/track/pdf/10.1186/s12864-018-5360-z">PDF</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30594132">PubMed</a>
<a href="https://doi.org/10.1101/428318">bioRxiv</a>
<a href="https://pub.uni-bielefeld.de/record/2932870">PUB</a>
<a href="https://doi.org/10.17863/CAM.34763">Apollo</a>
<a href="https://pub.uni-bielefeld.de/record/2931315">data</a>
<a href="https://github.com/bpucker/ncss2018">github</a>
</li>

<li>
Schilbert, H.M., Pellegrinelli, V., Rodriguez-Cuenca, S., Vidal-Puig, A. & <b>Pucker, B.</b> (2018). 
Harnessing natural diversity to identify key amino acid residues in prolidase. 
bioRxiv 423475; doi:10.1101/423475.<br>
<a href="https://doi.org/10.1101/423475">full text</a>
<a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/24/423475.full-text.pdf">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2931225">PUB</a>
<a href="https://github.com/bpucker/PEPD">github</a>
<a href="https://doi.org/10.5281/zenodo.2573170">zenodo</a>
</li>

<li>Brinkrolf, C., Henke, N. A., Ochel, L., <b>Pucker, B.</b>, Kruse, O., and Lutter, P. (2018).
Modeling and Simulating the Aerobic Carbon Metabolism of a Green Microalga Using Petri Nets and New Concepts of VANESA.
Journal of Integrative Bioinformatics 15. doi:10.1515/jib-2018-0018.<br>
<a href="https://doi.org/10.1515/jib-2018-0018">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://www.degruyter.com/downloadpdf/j/jib.2018.15.issue-3/jib-2018-0018/jib-2018-0018.xml">PDF</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30218605">PubMed</a>
<a href="https://pub.uni-bielefeld.de/record/2931064">PUB</a>
<a href="http://agbi.techfak.uni-bielefeld.de/vanesa">VANESA</a>
</li>

<li>Whitford, C. M., Dymek, S., Kerkhoff, D., M&auml;rz, C., Schmidt, O., Edich, M., Droste, J., <b>Pucker, B.</b>, R&uuml;ckert, C. and Kalinowski, J. (2018). 
Auxotrophy to Xeno-DNA: an exploration of combinatorial mechanisms for a high-fidelity biosafety system for synthetic biology applications. 
Journal of Biological Engineering, 12(1). doi:10.1186/s13036-018-0105-8.<br>
<a href="https://rdcu.be/4u5u">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://jbioleng.biomedcentral.com/track/pdf/10.1186/s13036-018-0105-8">PDF</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30123321">PubMed</a>
<a href="https://pub.uni-bielefeld.de/record/2930296">PUB</a>
<a href="https://doi.org/10.17863/CAM.26216">Apollo</a>
<a href="https://igem.org/Main_Page">iGEM</a>
</li>

<li>Haak, M., Vinke, S., Keller, W., Droste, J., R&uuml;ckert, C., Kalinowski, J., & <b>Pucker, B.</b> (2018).
High Quality <i>de novo</i> Transcriptome Assembly of <i>Croton tiglium</i>.
Frontiers in Molecular Biosciences, 5. doi:https://doi.org/10.3389/fmolb.2018.00062.<br>
<a href="https://doi.org/10.3389/fmolb.2018.00062">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6041412/pdf/fmolb-05-00062.pdf">PDF</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30027092">PubMed</a>
<a href="https://pub.uni-bielefeld.de/record/2921026">PUB</a>
<a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA416498">RNA-Seq data</a>
<a href="https://www.ncbi.nlm.nih.gov/nuccore/GGDV00000000.1">assembly</a>
</li>

<li>Friedrich, A., & <b>Pucker, B.</b> (2018). 
Peer-review as a teaching method. Working Paper der AG Forschendes Lernen in der dghd, 2, 2018. 
Carl von Ossietzky Universit&auml;t Oldenburg. <br>
<a href="https://pub.uni-bielefeld.de/download/2920633/2920815">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://pub.uni-bielefeld.de/download/2920633/2920815">PDF</a>
<a href="https://pub.uni-bielefeld.de/record/2920633">PUB</a>
<a href="https://github.com/bpucker/APPLS">github</a>
</li>

<li><b>Pucker, B.</b>, Holtgr&auml;we, D., & Weisshaar, B. (2017). 
Consideration of non-canonical splice sites improves gene prediction on the <i>Arabidopsis thaliana</i> Niederzenz-1 genome sequence.
BMC Research Notes, 10, 667. doi:10.1186/s13104-017-2985-y.<br>
<a href="https://doi.org/10.1186/s13104-017-2985-y">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://bmcresnotes.biomedcentral.com/track/pdf/10.1186/s13104-017-2985-y">PDF</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/29202864">PubMed</a>
<a href="https://pub.uni-bielefeld.de/record/2915524">PUB</a>
</li>

<li>Karsten, L., Bergen, D., Drake, C., Dymek, S., Edich, M., Haak, M., Kerkhoff, D., Kerkhoff, Y., Liebers, M., M&auml;rz, C., Schl&uuml;ter, L., Schmidt, O., Vinke, S., Whitford, C.M., <b>Pucker, B.</b>, Droste, J., R&uuml;ckert, C., M&uuml;ller, K.M., Kalinowski, J. (2017). 
Expanding The Genetic Code. Bielefeld University. doi:10.13140/RG.2.2.20342.91203.<br>
<a href="https://www.researchgate.net/publication/321586905_Expanding_The_Genetic_Code?channel=doi&linkId=5a27ea640f7e9b71dd0cca08&showFulltext=true">full text</a>
<a href="https://pub.uni-bielefeld.de/record/2930821">PUB</a>
</li>

<li>Alkhateeb, R., R&uuml;ckert, C., Rupp, O., <b>Pucker, B.</b>, Hublik, G., Wibberg, D., Niehaus, K., P&uuml;hler, A., Vorh&ouml;lter, F.J. (2017). 
Refined annotation of the complete genome of the phytopathogenic and xanthan producing <i>Xanthomonas campestris</i> pv. campestris strain B100 based on RNA sequence data.
Journal of Biotechnology. doi:10.1016/j.jbiotec.2017.05.009.<br>
<a href="https://doi.org/10.1016/j.jbiotec.2017.05.009">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/28506932">PubMed</a>
<a href="https://pub.uni-bielefeld.de/record/2911462">PUB</a>
</li>

<li><b>Pucker, B.</b>, Holtgr&auml;we, D., Rosleff S&ouml;rensen, T., Stracke, R., Vieh&ouml;ver, P., and Weisshaar, B. (2016).
A <i>de novo</i> Genome Sequence Assembly of the <i>Arabidopsis thaliana</i> Accession Niederzenz-1 Displays Presence/Absence Variation and Strong Synteny.
PloS-ONE 11:e0164321. doi:10.1371/journal.pone.0164321.<br>
<a href="https://doi.org/10.1371/journal.pone.0164321">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0164321&type=printable">PDF</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/27711162">PubMed</a>
<a href="https://pub.uni-bielefeld.de/record/2905844">PUB</a>
<a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA302255">NCBI BioProject</a>
</li>

<li>Frommer, B, Hain, C., Kampa, J., K&ouml;nigs, C., Linder, M., Perez Knoche, S., Sahin, M., Schmidt, P., Sch&ouml;ller, M., Kalinowski, J., M&uuml;ller, K.M., Droste, J., Hoffmann, N., Roeloffs, F., <b>Pucker, B.</b> (2016).
Generating Binding Proteins - via <i>in vivo</i> directed evolution in <i>E. coli</i>. doi:10.13140/RG.2.2.23164.95367.<br>
<a href="https://www.researchgate.net/publication/310965263_Generating_Binding_Proteins_-_via_in_vivo_directed_evolution_in_E_coli">full text</a>
</li>

<li>Fust A., Hollmann B., <b>Pucker, B.</b>, Wollborn D., Tiemann J., Droste J., Brosda S., Blunk S., Riedl S., Bleckwehl T. (2014).
The Transformers. From Carbon Dioxide to Biofuel. doi:10.13140/RG.2.2.30504.98566.<br>
<a href="https://www.researchgate.net/publication/287330796_iGEM_Bielefeld-CeBiTec_2014">full text</a>
<a href="https://pub.uni-bielefeld.de/record/2905012">PUB</a>
<a href="https://blogs.plos.org/collections/igem-report-006/">PLOS Report</a>
</li>

<li>Stracke, R., Holtgr&auml;we, D., Schneider, J., <b>Pucker, B.</b>, S&ouml;rensen, T.R., and Weisshaar, B. (2014).
Genome-wide identification and characterisation of R2R3-MYB genes in sugar beet (<i>Beta vulgaris</i>).
BMC Plant Biol. 14: 249. doi:10.1186/s12870-014-0249-8.<br>
<a href="https://doi.org/10.1186/s12870-014-0249-8">full text<font color="green">(peer-reviewed)</font></a>
<a href="https://bmcplantbiol.biomedcentral.com/track/pdf/10.1186/s12870-014-0249-8">PDF</a>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/25249410">PubMed</a>
<a href="https://pub.uni-bielefeld.de/record/2696778">PUB</a>
<a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA254489">RNA-Seq data</a>
</li>

</ul>

